<html>

<head>

    <link rel="stylesheet" type="text/css"
        href="https://2021.igem.org/wiki/index.php?title=Template:TEC_COSTA_RICA/BootstrapMinCSS&action=raw&ctype=text/css" />

    <!-- Load Vue followed by BootstrapVue -->
    <script
        src="https://2021.igem.org/wiki/index.php?title=Template:TEC_COSTA_RICA/VueJS&action=raw&ctype=text/javascript">
    </script>

    <script
        src="https://2021.igem.org/wiki/index.php?title=Template:TEC_COSTA_RICA/BootstrapJS&action=raw&ctype=text/javascript">
    </script>

    <link href='https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,800;1,800&display=swap' rel='stylesheet'>

</head>


<!-- COMPONENTE CAROUSEL -->
<style>
    .carousel .carousel-caption {
        padding-top: 2rem;
        height: 100%;
    }

    .carousel .carousel-caption h3 {
        font-size: 50px;
        color: white !important;

    }

    .carousel .carousel-caption p {
        /*font-size: 12px;*/
    }

    .page_content_sections .page_content_section {
        display: none;
        padding: 10%;
        padding-top: 20px;
    }

    .page_content_sections .page_content_section_active {
        display: block;
    }
</style>

<script type="text/x-template" id="main_carousel">
    <div>
        <div>
            <div :id="'carouselComponent'+id" class="carousel slide" data-bs-ride="carousel" data-bs-interval="false"  ref="maincarousel"  >


                <ol class="carousel-indicators">
                    <li v-for="(slide, index) in slides_content" :data-bs-target="'#carouselComponent'+id"  v-bind:data-bs-slide-to="index" v-bind:class="{ active: index==0 }"></li>
                </ol>

                <div class="carousel-inner" role="listbox">


                    <div class="carousel-item" v-for="(slide, index) in slides_content"  v-bind:class="{ active: index==0 }">
                        <svg class="bd-placeholder-img" width="100%" height="250px" xmlns="http://www.w3.org/2000/svg"
                            aria-hidden="true" preserveAspectRatio="xMidYMid slice" focusable="false">
                            <rect width="100%" height="100%" fill="#1A1A1A" />
                        </svg>

                        <div class="carousel-caption text-start">
                            <h3 class="slide_tittle">
                                {{slide.title}}

                            </h3>
                            <p class="slide_text w-75">
                                {{slide.content}}
                            </p>
                        </div>
                    </div>
            </div>


            <button class="carousel-control-prev" type="button" :data-bs-target="'#carouselComponent'+id"
                data-bs-slide="prev">
                <span class="carousel-control-prev-icon" aria-hidden="true"></span>
                <span class="visually-hidden">Previous</span>
            </button>
            <button class="carousel-control-next" type="button" :data-bs-target="'#carouselComponent'+id"
                data-bs-slide="next">
                <span class="carousel-control-next-icon" aria-hidden="true"></span>
                <span class="visually-hidden">Next</span>
            </button>

        </div>
    </div>       
</script>


<script>
    Vue.component("main-carousel", {
        data: {},
        props: {
            current_slide: {
                type: Object,
                default: "casa"
            },
            slides_content: {
                type: Array,
                default: []
            },
            id: {
                type: Number,
                default: 0
            }
        },

        template: "#main_carousel",
        methods: {
            updateCurrentSlide: function (event) {
                this.current_slide.index = event.to;
            },

        },
        mounted() {
            this.$refs.maincarousel.addEventListener('slide.bs.carousel', this.updateCurrentSlide);
        }
    });
</script>
<!-- COMPONENTE CAROUSEL -->




<!-- CONTENIDO PRINCIPAL -->

<body>
    <div id="app">

        <div class="position-relative overflow-hidden pt-5 px-5 text-start  main-title">
            <img class="main-title-bg" src="https://picsum.photos/id/1042/1200/400" alt="">
            <h1>Description</h1>
            <h5 class="">Our project consists of two phases. The first encapsulates the  circuit and focuses on the fulfillment of specific requirements through an in-depth architectural design along with a thorough assessment of the circuit’s behavior and characteristics. The second focuses on a conceptual framework: a complete guide for the assessment of sequential logic in synthetic biology, together with its implementation as a model for our software tool’s design. </h5>

            <div class="nav nav-pills  justify-content-start mt-4 mb-2 " id="nav-tab-main" role="tablist">
                <button class="btn btn-light btn-lg  active mx-2 wiki-tab-btn" id="nav-circuit-tab" data-bs-toggle="tab"
                    data-bs-target="#pills-circuit" type="button" role="tab" aria-controls="pills-circuit"
                    aria-selected="true">Circuit</button>
                <button class="btn btn-light btn-lg  mx-2 wiki-tab-btn" id="nav-framework-tab" data-bs-toggle="tab"
                    data-bs-target="#pills-framework" type="button" role="tab" aria-controls="pills-framework"
                    aria-selected="false">Framework</button>
                <!-- <button class="btn btn-light btn-lg  mx-2 wiki-tab-btn" id="nav-other-tab" data-bs-toggle="tab"
                            data-bs-target="#pills-other" type="button" role="tab" aria-controls="pills-other"
                            aria-selected="false">Other</button> -->
            </div>
        </div>

        <!-- DEMO TABS -->
        <div class="">

            <div class="tab-content" id="pills-tabContent">



                <!-- pills-circuit -->
                <div class="tab-pane fade show active" id="pills-circuit" role="tabpanel"
                    aria-labelledby="pills-circuit-tab">


                    <main-carousel v-bind:current_slide="current_slide_index['circuit']"
                        v-bind:slides_content="slides_content['circuit']" :id="0">
                    </main-carousel>


                    <!-- Introduction -->

                    <div v-if="current_slide_index['circuit'].index==0" class="container">

                        <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                            <div class="col-sm-6">
                                <!-- <h2>title </h2> -->
                                <!-- <h3>subtitle </h3> -->
                                <p>
                                    For the design of said circuit we defined three essential needs: <br><br>

                                    1. A way to <b>determine the lifetime</b> of our engineered bacteria<br>

                                    2. A mechanism to <b>prevent</b> the life span from running out during laboratory manipulation<br>

                                    3. A multilayered <b>killswitch</b> to ensure a safe death<br>

                                </p>
                            </div>

                            <div class="col-sm-6 justify-content-md-center">
                                <img class="img-square align-middle " src="https://2021.igem.org/wiki/images/f/ff/T--TEC_COSTA_RICA--Description_GSC_12_Bacteria.png"
                                    all="">
                            </div>
                        </div>


                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <!-- <h2>title </h2> -->
                            <!-- <h3>subtitle </h3> -->

                            <p class="">
                                We tackled our first design requirement by exhaustive brainstorming and research. Our main idea involved a <b>counter system</b>, hence, multiple mechanisms to perform this count were discussed, from binary linear systems dependent on inducible promoters to our final implementation, a <b>sequential logic based recombinase device</b>. Some of the counter systems we assessed and used as inspiration are in the {comparison} section. <br><br>

                                In spite of the value of the previously mentioned proposals, none of the counters fit our more specific needs which were focused on the complexity, count limit and length of the device. Since we envisioned our circuit being used as an <b>auxiliary</b> to other applications, it was essential that the available count was as high as possible without the length growing exponentially. This left out counters such as the ones proposed by Groningen Team (2011) and Paris Liliane Bettencourt Team (2010), since they behave linearly and “in situations where the same signal is being recorded over multiple occurrences (for example, a series of cell division events), reliably rewritable elements are needed to realize geometric increases in data storage capacity (for example, combinatorial counters capable of recording 2N events given N storage elements)” (ref). Whereas Team: UT-Tokyo’s (2016) approach was not only lengthy but also complex and imposing a high metabolic burden, which were undesired qualities as well. Other proposals, even with 2n capability, displayed undesirable qualities such as the need for very specific regulatory systems such as the delay system presented by Zhao et al. (2019). Also, many of the counters were based on the theoretical assumption that there are n protein-induced promoters or n serine recombinases which don't have crosstalk issues being available, stating the limit at the number of orthogonal characterized recombinases (Yang et al., 2014). While good for stating our counter’s basis, such as the viability of placing several recombinases next to each other and having them behave correctly, we wanted to go even further. This led us to propose our own counter system, which went through several changes and iterations as explained in {engineering success} until we came up with a suitable approach as validated by <b>PhD. Pakpoom Subsoontorn and PhD. Jérôme Bonnet</b>. <br><br>

                                It’s important to note that our counter is a <b>practical representation</b> and <b>proof of concept</b> for the sequential-logic based development of genetic circuits that we propose in the {conceptual framework}. <br><br>

                                The final genetic suicide circuit design consists of different modules, whose functions are explained next.

                            </p>
                        </div>

                       

                        <div class="container  mx-auto p-3 wikicard-12-img">
                                    <h1>
                                        title
                                    </h1>
                                <div class="text-center">
                                        <img src="https://2021.igem.org/wiki/images/7/7c/T--TEC_COSTA_RICA--Description_GSC_Introduction_SS.png" alt="" style="width:100% center" class="w-75">
                                </div>
                        </div>
                        

                    </div>




                    <!-- Function -->
                    <div v-if="current_slide_index['circuit'].index==1" class="container">

                        <div class="container wiki-card mx-auto p-3">
                            <div class="row">
                                <div class="col-sm-1 text-end">
                                    <h1>
                                        01
                                    </h1>
                                </div>
                                <div class="col-sm-7">
                                    <h2>Repress-Initialize</h2>

                                    <p>
                                        The first device was designed to maintain the other three <b>inactivated</b> until the user is ready to trigger the countdown with the intent of permitting the manipulation, cultivation, etc. of the engineered organism without the circuit counter being activated. Otherwise, if the circuit wasn’t under strict control, the growing times and uses of these microorganisms would be limited. The proposed mechanism consists of regulating the transcription of the main promoter via repression triggered by an inducible promoter.
                                    </p>
                                </div>
                                <div class="col-sm-4  ">
                                    <img src="https://placehold.co/300x300?text=snippet" class="img-round align-middle"
                                        all="">
                                </div>
                            </div>
                        </div>

                        <div class="container wiki-card mx-auto p-3">
                            <div class="row">
                                <div class="col-sm-4  ">
                                    <img src="https://placehold.co/300x300?text=snippet" class="img-round align-middle"
                                        all="">
                                </div>
                                <div class="col-sm-1 text-end">
                                    <h1>
                                        02
                                    </h1>
                                </div>
                                <div class="col-sm-7">
                                    <h2>Count</h2>

                                    <p>
                                        Autonomous genetic counter whose purpose is to keep track of the number of <b>times</b> a cell goes through cell division (from the moment the counter is activated by the first device). The counter is based on a cell-cycle coupled promoter, recombinases and unidirectional terminators, as detailed in the {architecture section}. This device is able to count up to <b>2n</b> where n is the number of recombinases.
                                    </p>
                                </div>
                            </div>
                        </div>

                        <div class="container wiki-card mx-auto p-3">
                            <div class="row">
                                <div class="col-sm-1 text-end">
                                    <h1>
                                        03
                                    </h1>
                                </div>
                                <div class="col-sm-7">
                                    <h2>Expand Count</h2>

                                    <p>
                                        The third device’s purpose is to enable a <b>higher count</b>, hence it’s also regulated by the second device’s promoter and it’s based on a self-excising DNA segment that <b>resets</b> the counter state of the previous device. This system is CRISPR Cas-9 reliant and amplifies the count by x times where x is the number of repeats of self-excising segments. 
                                    </p>
                                </div>
                                <div class="col-sm-4  ">
                                    <img src="https://placehold.co/300x300?text=snippet" class="img-round align-middle"
                                        all="">
                                </div>
                            </div>
                        </div>

                        <div class="container wiki-card mx-auto p-3">
                            <div class="row">
                                <div class="col-sm-4  ">
                                    <img src="https://placehold.co/300x300?text=snippet" class="img-round align-middle"
                                        all="">
                                </div>
                                <div class="col-sm-1 text-end">
                                    <h1>
                                        04
                                    </h1>
                                </div>
                                <div class="col-sm-7">
                                    <h2>Kill Switch</h2>

                                    <p>
                                        Multi-layered containment system designed with the objective of <b>killing the host</b> cells leaving no trace and whose initiation depends on previous devices. Several toxins are involved in this device, including RelE and NucA1, along with self-targeting single guides to ensure its safety.
                                    </p>
                                </div>
                            </div>
                        </div>






                    </div>

                    <div v-if="current_slide_index['circuit'].index==2" class="container">

                        <div class="container  mx-auto p-3 ">
                            <p>In the following image you can see the design of our whole device and, by clicking on the
                                boxes below, you can access our insight on each of the different parts and modules.
                            </p>
                            <div class="text-center">
                                <img src="https://2021.igem.org/wiki/images/7/78/T--TEC_COSTA_RICA--Description_GSC_11_SuicideCircuit_Architecture.png" alt="" style="width:100% center"
                                    class="w-75">
                            </div>
                        </div>



                        <nav>
                            <div class="nav nav-pills  justify-content-center py-4 " id="nav-tab" role="tablist">

                                <button class="btn btn-lg btn-outline-primary active mx-2 my-2" id="nav-promoters-tab"
                                    data-bs-toggle="tab" data-bs-target="#pills-promoters" type="button" role="tab"
                                    aria-controls="pills-promoters" aria-selected="true"
                                    style=" border-radius: 50px;">Promoters </button>

                                <button class="btn btn-lg btn-outline-primary mx-2 my-2 " id="nav-recombinases-tab"
                                    data-bs-toggle="tab" data-bs-target="#pills-recombinases" type="button" role="tab"
                                    aria-controls="pills-recombinases" aria-selected="false"
                                    style=" border-radius: 50px;">Recombinases</button>

                                <button class="btn btn-lg btn-outline-primary mx-2 my-2 " id="nav-terminator-tab"
                                    data-bs-toggle="tab" data-bs-target="#pills-terminator" type="button" role="tab"
                                    aria-controls="pills-terminator" aria-selected="false"
                                    style=" border-radius: 50px;">Terminator</button>

                                <button class="btn btn-lg btn-outline-primary mx-2 my-2 " id="nav-crispr-tab"
                                    data-bs-toggle="tab" data-bs-target="#pills-crispr" type="button" role="tab"
                                    aria-controls="pills-crispr" aria-selected="false"
                                    style=" border-radius: 50px;">CRISPR+Ligase</button>

                                <button class="btn btn-lg btn-outline-primary mx-2 my-2 " id="nav-operator-tab"
                                    data-bs-toggle="tab" data-bs-target="#pills-operator" type="button" role="tab"
                                    aria-controls="pills-operator" aria-selected="false"
                                    style=" border-radius: 50px;">Operator</button>

                                <button class="btn btn-lg btn-outline-primary mx-2 my-2 " id="nav-multilay-tab"
                                    data-bs-toggle="tab" data-bs-target="#pills-multilay" type="button" role="tab"
                                    aria-controls="pills-multilay" aria-selected="false"
                                    style=" border-radius: 50px;">Multi layered containment system</button>

                                <button class="btn btn-lg btn-outline-primary mx-2 my-2 " id="nav-other-tab"
                                    data-bs-toggle="tab" data-bs-target="#pills-other" type="button" role="tab"
                                    aria-controls="pills-other" aria-selected="false"
                                    style=" border-radius: 50px;">Other elements</button>
                            </div>

                        </nav>

                        <div class="tab-content" id="pills-tabContent">

                            <div class="tab-pane fade show active" id="pills-promoters" role="tabpanel"
                                aria-labelledby="pills-promoters-tab">


                                <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                                    <div class="col-sm-6">
                                        <h2>Promoters </h2>
                                        <p>
                                            Promoters are the main regulatory units for <b>translation</b>; they define where translation<b> starts</b> and under which conditions (iGEM, s.f.a). 
                                        </p>
                                    </div>

                                    <div class="col-sm-6">
                                        <h3>Promoter 1: pBAD {BBa_I0500} </h3>
                                        <p>
                                            This promoter is induced by <b>arabinose</b> and it’s one of the most characterized for heterologous protein expression (Széliová et al., 2016), and commonly used in iGEM and synthetic biology itself (http://parts.igem.org/PBAD_Promoter_Family http://parts.igem.org/Part:BBa_I0500). Many of the other counter systems discussed in this project were developed and characterized with this promoter as its regulatory element (Roquet et al., 2016; Bonnet et al., 2012; Zhao et al., 2019; Yang et al., 2014). 
                                        </p>
                                    </div>
                                </div>

                                <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                                    <div class="col-sm-6">
                                        <h3>Promoter 2: pnrd {BBa_K2070012} </h3>
                                        <p>
                                            This promoter is <b>cell cycle dependent</b> according to (Sun 1992), since it is a highly regulated promoter region which contains several DnaA boxes, indicating that activity is partly affected by initiation of DNA replication (Messer 2002). The nrd operon itself in E. coli expresses a ribonucleotide reductase, an enzyme that reduces ribonucleotides into deoxyribonucleotides and is involved in bacterial cell cycle in E. coli (Sun 1994).  This part has been used previously by Stanford Brown in 2012 (http://2012.igem.org/Team:Stanford-Brown); they proved that the promoter turned on once every cell cycle. A variation of this promoter was also used by UT-Tokyo 2016 (link) to build a regulatory system in which gene expression changes after each cell division which is very similar to our own purpose. The use of this promoter ensures that our system will be activated only once every cell cycle. 
                                        </p>
                                    </div>

                                    <div class="col-sm-6 justify-content-md-center">
                                        <img class="img-square align-middle "
                                            src="https://placehold.co/600x600?text=snippet" all="">
                                    </div>
                                </div>



                            </div>


                            <div class="tab-pane fade show " id="pills-recombinases" role="tabpanel"
                                aria-labelledby="pills-recombinases-tab">



                                <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                                    <h2>Recombinases </h2>

                                    <p class="">
                                        DNA site-specific recombinases are enzymes that catalyse breaking and rejoining of DNA strands at specific points, thereby bringing about precise genetic <b>rearrangements</b>. Serine integrases are a group of recombinases with unusual properties such as recombination <b>directionality</b> and simple site requirements (Stark, 2017). According to Zhao et al. (2019) when inverting, serine recombinases recognize specific genetic sites, attB and attP (PB state), and invert the sequence between these sites generating the new sites attL and attR (LR state). This inversion is unidirectional, meaning that another component must be present for the inversion to be reversed. This component is the <b>RDF</b> (recombination directionality factor), and, when co-expressed with the recombinase, allows to switch the LR state back to PB (Zhao et al., 2019). Depending on the DNA arrangement, the use of integrases allow the control of the flow of RNA polymerase along the DNA, hence <b>regulating genetic expression</b> (Merrick et al., 2018). The selected recombinases have been tried out together and display orthogonality (Guiziou et al., 2019). They have also been characterized in several other chassises, making our circuit easier to extrapolate (Tomimatsu et al., 2017; Xu & Brown, 2016).  BXB1 and TP901 particularly have been used in similar proposals such as Hsiao et al. (2016) y Roquet et al. (2016). 
                                 
                                        It’s important to highlight that the order assigned to these recombinases is not random, and was supported by our mathematical model. 
                                    </p>
                                </div>

                                <div class="container  mx-auto p-3 wikicard-12-img">

                                    <div class="text-center">
                                        <img src="//placehold.co/1200x600?text=snippet" alt="" style="width:100% center"
                                            class="w-75">
                                    </div>
                                </div>

                                <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                                    <div class="col-sm-6">
                                       <h3>Recombinase 1:Bxb1 </h3>
                                       
                                        <p>
                                            We chose Bxb1 as the first integrase since it displayed the best recombination efficiency</b> to obtain products with no residual substrates remaining as reported by Wang et al.  (2017), showing that the activity of Bxb1 integrase is obviously superior compared with any other integrase. Bxb1 integrase mediated excellent, efficient site-specific recombination in vitro in this study.
                                        </p>
                                    </div>

                                    <div class="col-sm-6">
                                        <h3>Recombinase 2, 3 and 4: 
                                        Represent TP901, Int5 and Int7 respectively. </h3>
                                        <p>
                                            There are several versions of each recombinase and other recombinases available in the registry http://parts.igem.org/Recombination/Other 
                                        </p>
                                    </div>
                                </div>



                            </div>




                            <div class="tab-pane fade show " id="pills-terminator" role="tabpanel"
                                aria-labelledby="pills-terminator-tab">


                                <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                                    <h2>Terminator </h2>

                                    <p class="">
                                        Terminators define the <b>end of translation</b>, they can be uni or bidirectional and are relevant to correct protein expression (iGEM, s.f.b).  
                                    </p>
                                </div>

                                <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                                    <div class="col-sm-6">
                                        <h3>Terminator 1: 
                                      {BBa_B1002} </h3>
                                        <p>
                                             <b>Bidirectional</b> terminators to ensure the separation of the different modules, i.e. 
                                        </p>
                                    </div>

                                    <div class="col-sm-6">
                                        <h3>Terminator 2: 
                                        {BBa_B0010} </h3>
                                        <p>
                                            Only functions in its <b>forward</b> direction and can be used to regulate translation of polycistronic elements similarly to Friedland et al. (2009)’s proposal, in which they built a DNA Invertase Cascade (DIC).
                                        </p>
                                    </div>
                                </div>





                            </div>


                            <div class="tab-pane fade show " id="pills-crispr" role="tabpanel"
                                aria-labelledby="pills-crispr-tab">



                                <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                                    <h2>CRISPR+ligase </h2>

                                    <p class="">
                                        The CRISPR-Cas9 system, a gene editing system, depends on two main elements, a small RNA fragment that contains the guide sequence that binds to the target DNA sequence and the Cas9 enzyme joined to the guide RNA sequence. When the guide is transcribed, it binds to the Cas9 in order to recognize the target DNA sequence which is then cut by the enzyme, creating a double strand  break. This break can be used to delete, insert or replace DNA if used coupled with the cell’s own repair machinery (MedlinePlus, s.f.). In our case E. coli doesn’t have a non-homologous end joining (NHEJ) repair system, so a double stranded break can only be fixed by homologous recombination. Since we don’t want the insertion of a template to be required, we searched for a different alternative, and found that a T4 ligase can fix the break creating indels that <b>knockout</b> the gene and prevent the cell from committing suicide because of the broken DNA (Su et al., 2016). This is according to Su et al. (2019), who report using a ligase for repairing a DSB made by Cas9 in LacZ gene in E. coli, leading to the survival of cells. In turn Su et al. (2016) report using a different strategy: which involved the design of sg to avoid HR and the use of conserved prokaryotic NHEJ proteins from Mycobacterium tuberculosis H37Rv instead for repairing complete gene deletions, proving that this is in fact possible even though in their case it had only 36% efficiency. We believe that by joining the best of each strategy, the ligase and the sg design, and having a much smaller deletion target (which in turn is closer to DNA deletion consistency at 200bp) we can have a much better efficiency in both deletion and repair.
                                    </p>
                                </div>


                                <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                                    <h3>Cas9 </h3>
                                    <p class="">
                                         http://parts.igem.org/Part:BBa_K1774001:Experience
                                    </p>

                                    <h3>Sg </h3>
                                    <p class="">
                                        Our single guides are to be designed to fulfill several requirements for an increased specificity and efficiency and decreased off-target effects as proposed by Liu et al. (2020). 
                                    </p>

                                    <h3>Ligase </h3>
                                    <p class="">
                                        {BBa_K3917004},  created by basing off of Su et al. (2019). 
                                    </p>
                                </div>


                            </div>


                            <div class="tab-pane fade show " id="pills-operator" role="tabpanel"
                                aria-labelledby="pills-operator-tab">


                                <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                                    <h2>Operator sites </h2>

                                    <p class="">
                                        “A common method of <b>repressing gene expression</b> in prokaryotes involves the binding of a protein to a DNA sequence (a.k.a. operator) near or overlapping its promoter” (Politz et al., 2013). These operators, in our design, were purposely ambiguous, since their identity can vary depending on available technology. The common requirement they must fulfill is binding to specific DNA sequences. A readily available mechanism to do this is taking advantage of already existing repressors such as LacI from the Lac operon or the phage encoded lambda repressor (cI) (Lewis, 2011). The deactivated version of the cas9 protein is also an alternative, but its location would have to be rethought, since it would need to be active and functional for all three cases (Xu & Qi, 2019). Finally, other synthetically designed DNA-binding proteins can be used, TALEs being one of the most used of these categories (Politz et al., 2013). 
                                    </p>
                                </div>

                            </div>



                            <div class="tab-pane fade show " id="pills-multilay" role="tabpanel"
                                aria-labelledby="pills-multilay-tab">

                                <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                                    <h2>Multi-layered containment system </h2>
                                    
                                    <p class="">
                                        
                                        A multi-layered safeguard system is a series of barriers that can decrease the escape frequency of synthetically modified microorganisms (Gallagher et al., 2015). In this device, the main objective is to develop a biocontainment, condition independent, kill switch.<br><br>
                                    </p>
                                    
                                    <h3>RelE: modified {BBa_K2449029}, removing the double terminator. </h3>

                                    <p class="">
                                        The toxin RelE, a mRNA interferase, causes <b>translation inhibition</b> by cutting mRNAs positioned on site A of the ribosome. It cleaves the stop codons of the mRNA between the 2nd and 3rd nucleotide (https://www.uniprot.org/uniprot/P0C077), which blocks processes such as its binding to release factors, release of formed polypeptides and availability of ribosomes to be used again (SJTU-BioX-Shanghai, 2009). This ultimately hinders the correct translation and promotes a decrease in cell growth. However, since mRNA interferases play a role in bacterial persistence to antibiotics (https://www.uniprot.org/uniprot/P0C077), it is necessary to additionally apply other death mechanisms together. 
                                    </p>
                                </div>

                                <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                                    <h3>Nuclease A1: modified {BBa_K3027000}, removing its terminator. </h3>

                                    <p class="">
                                        The nuclease A1 is in charge of the <b>genomic DNA elimination</b>, without being associated with cell lysis (GO_Paris-Saclay, 2009), which is important because the genetic material can remain contained during the DNA escision. When the cell is left without DNA, some remnants, like ribosomes and housekeeping proteins, could still get transferred when interacting with other microorganisms and provide them new functions including and beyond our specific modification. DNA-less cells are still equivalent to cells with DNA (GO_Paris-Saclay, 2009) with the minimal genome to maintain metabolic homeostasis, reproduce, and evolve (Gil et al., 2004). 
                                    </p>
                                </div>


                                <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                                    <h3>CRISPR: GMK target </h3>

                                    <p class="">
                                        Already described mechanism of action (see CRISPR+ligase). In this module, it includes is a sgRNA to target the nucleoside monophosphate kinases <b>Gmk</b>, a multimeric enzyme showed to be determinant in the reduction of the nucleotide pool and decrement of growth and/or respiration rate when it is knocked out, due to the specificity established  with the phosphate acceptor substrate (EcoCyc, 2012). We also aim to use another set of sgRNA to generate breaks to our <b>synthetic construct</b>, leaving no possibilities for synthetic genomic remnants to be freed. Single guides are to be designed as specified previously, targeting the Gmk gene, the synthetic circuit’s main promoter and its recombinases. 
                                    </p>
                                </div>



                            </div>




                            <div class="tab-pane fade show " id="pills-other" role="tabpanel"
                                aria-labelledby="pills-other-tab">

                                <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                                    <h2>Plasmid </h2>

                                    <p class="">
                                        Heterologous expression in E. coli may be achieved through several vectors such as plasmids or bacterial chromosomes, as well as by the insertion of DNA directly into the bacterial artificial chromosome. The ideal implementation for our proposal is <b>chromosomal insertion</b>, in order to prevent the existence of cells with a <b>mixed expression</b> within them (Zhao et al, 2019), otherwise understood as several counters in a different state. It’s also a way to allow the stable expression of foreign DNA without the need for antibiotic selection, the increased protein expression and the decrease of metabolic burden (Englaender et al., 2017). The likelihood of uptake of the synthetic DNA by other organisms is also decreased by this strategy. Nonetheless if it’s to be implemented in a plasmid, this should be <b>low-copy</b> in order to reduce the noise caused from the plasmid’s divergent expression, and the selection mechanism should be via auxotrophy to allow for more diverse implementations.
                                    </p>
                                </div>


                                <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                                    <h2>Degradation tags and RBS optimization</h2>

                                    <p class="">
                                        For our system to be as efficient as possible all of the recombinase’s expressions must be fine-tuned. One of the most common ways to do this is via RBS selection, which would allow the <b>balancing out</b> of the recombinases and their expression rate by regulating their translation (Jin et al., 2017). Also, this way the ratio of 2:1  needed for correct RDF:recombinase expression can be achieved. Another relevant point for our system is that recombinases are to execute their work and then be degraded as quickly as possible to not interfere with other recombinases. This is particularly true for RDFs, since they’re being generated several generations previous to their participation. Hence, we researched ways to fast-teack the degradation of proteins, particularly the application of degradation tags, which are a mechanism for <b>targeted proteolysis</b> mediated by the addition of a degradation tag to the coding sequence (McGinness et al., 2006) . These have been used in counter systems to improve their efficiency such as that (SYSU China Team, 2015), and can similarly be applied to our circuit.
                                    </p>
                                </div>


                            </div>



                        </div>




                    </div>



                    <!-- caracteristicas -->
                    <div v-if="current_slide_index['circuit'].index==3" class="container">
                        <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                            <div class="col-sm-6">
                                <h2>Evolutionary Stability </h2>
                                <h3>Evolutionary stability of recombinase based memory circuits  </h3>
                                <p>
                                    According to Fernandez-Rodriguez et al. (2015) one of the design principles for long-term stability is the <b>avoidance of constitutive expression</b>, which we have applied in our circuit. They also mention that “Placing the circuit in the <b>genome</b> could also improve stability, as this is a common approach to improve the robustness of strains in metabolic engineering” (Fernandez-Rodriguez et al., 2015). Another consideration that must be made is the <b>prevention of leaky expression</b>, since it can generate a drift into a different state and even disable the ability to hold a state. In our circuit the expression of all recombinases is regulated by the same promoter, so this issue is not expected to be relevant to its performance. Canton et al., (2008) note that the evolutionary stability is also dependent on the <b>state</b> in which the circuit is carried; the authors note that a circuit in the OFF state can easily be carried over many generations, whereas in the ON state it quickly breaks. This specific quality can be problematic for our circuit, since one way or another it’s always turned on because of the dependence on the cell-cycle promoter. Nonetheless, since different activities are carried out in each specific state and there is no constant expression of regulatory proteins, this might not be relevant to the circuit’s stability, since “the burden is a function of regulator expression, the evolutionary stability depends on the state in which the circuit is carried” (Fernandez-Rodriguez et al., 2015). <br><br>

                                    This was seconded by Sleight et al. (2010), who state that “Evolutionary stability is a problem in genetic circuits if there is no selective pressure to maintain function of the circuit. The current belief is that this loss-of-function occurs because any cell in the population that acquires a mutation in the genetic circuit often has a growth advantage and can outcompete the cells in the population with all functional plasmids”; hence, if there is no <b>significant metabolic burden</b>, there is no advantage.  Finally, we weren’t able to carry out an evolutionary stability experiment of our own; however, extrapolating the results obtained by Fernandez-Rodriguez et al. (2015) shows great promise for our circuit. They report a stability of over 400 hours for their memory switch, and credit its breaking to continuous invertase expression and the host’s evolution towards reducing its expression. More importantly, Fernandez-Rodriguez et al. (2015) indicate that during the 400 hour period, 6 cycles of their circuit, “there was no loss of performance, reduction in dynamic range, or increase in population variability”. The main risk to our circuits stability are transposable insertion elements and even homologous recombination, which can be avoided by using <b>diverse parts</b> in the construction of the circuit and using strains where insertion elements have been deleted from the genome; nevertheless, both these strategies involve challenges on themselves, particularly the need for more robust strains for many applications, as stated by Fernandez-Rodriguez et al. (2015). Finally, although evolutionary stability must be considered when designing a circuit, the main focus should be on the specific end application. The required robustness of a circuit will be directly related to the <b>duration of its application</b>. “For example, circuits do not need to be robust for months if designed for a one week fermentation”(Fernandez-Rodriguez et al., 2015). <br><br>

                                    The time for the loss of function of a genetic circuit can also be predicted by applying the <b>function</b> developed by Arkin & Fletcher (2006) through a simulation study. 
                                </p>
                            </div>

                            <div class="col-sm-6 justify-content-md-center">
                                <img class="img-square align-middle " src="https://placehold.co/600x600?text=snippet"
                                    all="">
                            </div>
                        </div>

                        <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                            <div class="col-sm-6">
                                <h3>Evolutionary stability of kill switches </h3>
                                <p>
                                    According to Stirling et al. (2017),  “any kill switch with leaky, low-level expression of a toxin in permissive conditions may be quickly disabled in rapidly growing microbes.” In our circuit this is highly unlikely, since the expression of the toxins is regulated by the same promoter that controls the expression of recombinases, and between them stand several terminators. Furthermore, the toxins from our kill-switch are to be expressed only once, and induced by the change of state from the previous module (counter circuit), hence <b>avoiding the complexities of an environmentally induced kill switch</b>. The necessity for induction is another decisive factor in the stability of the circuit, since, <b>uninduced there is no selective pressure for it to be dismantled</b>. This was proven by Pasotti et al. (2011), who evaluated the stability of an inducible killswitch, finding it to be stable for a maximum of 100 generations and strain dependent. <br><br>

                                    Further optimization for the final application can be done by applying tools such as “Predicting the Genetic Stability of Engineered DNA Sequences with the EFM Calculator” and making the necessary alterations regarding the sequence such as altering codon usage in the different CDSs or swapping out promoters/rbs of equal strength (Jack et al., 2015).<br><br>

                                    For further characterization and application examples, check out our Proposed Implementation page.
                                </p>
                            </div>

                            <div class="col-sm-6 justify-content-md-center">
                                <img class="img-square align-middle " src="https://placehold.co/600x600?text=snippet"
                                    all="">
                            </div>
                        </div>

                    </div>


                    <!-- functional animation -->
                    <div v-if="current_slide_index['circuit'].index==4" class="container">


                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <!-- <h2>title </h2> -->
                            <!-- <h3>subtitle </h3> -->

                            <p class="">
                                The functioning is based on four main premises based on the architecture:
                            </p>
                        </div>


                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <!-- <h2>title </h2> -->
                            <!-- <h3>subtitle </h3> -->

                            <p class="wiki-card-2columns">
                                <ul>
                                    <li>The promoter will work one time per cell cycle.</li>
                                    <li>The terminator will only work in its forward direction.</li>
                                    <li>Each recombinase (when expressed) will invert non-reversibly the DNA sequence when such is flanked by PB sites.</li>
                                    <li>Each recombinase together with the recombination directionality factor (RDF) (when both are expressed) will invert non-reversibly the DNA sequence when such is flanked by LR sites.</li>
                                </ul>
                            </p>
                        </div>




                        <div class="accordion accordion-flush" id="accordionPanelsStayOpenExample">



                            <div class="accordion-item">
                                <h2 class="accordion-header" id="panelsStayOpen-headingOne">
                                    <button class="accordion-button" type="button" data-bs-toggle="collapse"
                                        data-bs-target="#panelsStayOpen-collapseOne" aria-expanded="true"
                                        aria-controls="panelsStayOpen-collapseOne">
                                        Repress-Initialize
                                    </button>
                                </h2>
                                <div id="panelsStayOpen-collapseOne" class="accordion-collapse collapse show"
                                    aria-labelledby="panelsStayOpen-headingOne">
                                    <div class="accordion-body">



                                        <div class="container  mx-auto p-3 wikicard-12-img">

                                            <div class="text-center">
                                                <img src="https://2021.igem.org/wiki/images/8/8e/T--TEC_COSTA_RICA--Description_GSC_13_GIFinitialize.gif" alt=""
                                                    style="width:100% center" class="w-50">
                                            </div>
                                        </div>



                                        <div
                                            class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                                            <h2>Repress-Initialize </h2>
                                            <p class="">
                                                During lab work with the cloned bacteria, the inducer must be provided in order for the counter to be repressed. Once ready to implement the bacteria on its specific application the inducer must be withdrawn from or not supplied to the culture. The repressor will then be degraded and the counter will start.
                                            </p>
                                        </div>


                                    </div>
                                </div>
                            </div>



                            <div class="accordion-item">
                                <h2 class="accordion-header" id="panelsStayOpen-headingTwo">
                                    <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
                                        data-bs-target="#panelsStayOpen-collapseTwo" aria-expanded="false"
                                        aria-controls="panelsStayOpen-collapseTwo">
                                        Counter Circuit
                                    </button>
                                </h2>
                                <div id="panelsStayOpen-collapseTwo" class="accordion-collapse collapse"
                                    aria-labelledby="panelsStayOpen-headingTwo">
                                    <div class="accordion-body">

                                        <div class="container  mx-auto p-3 wikicard-12-img">

                                            <div class="text-center">
                                                <img src="https://2021.igem.org/wiki/images/b/b6/T--TEC_COSTA_RICA--Description_GSC_14_GIFContador.gif" alt=""
                                                    style="width:100% center" class="w-50">
                                            </div>
                                        </div>



                                        <div
                                            class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                                            <h2>Counter Circuit</h2>
                                            <h3> </h3>

                                            <p class="">
                                                
                                            </p>
                                        </div>



                                        <div class="container wiki-card  mx-auto p-3 row wikicard-6-2">
                                            <div class="col-sm-6 row wiki-card-6 ">
                                                <div class="col-sm-2 text-end">
                                                    <h1>
                                                        A
                                                    </h1>
                                                </div>
                                                <div class="col-sm-10">
                                                    <h2><b>First generation</b> </h2>
                                                    <p>
                                                        Only the first recombinase is transcribed and hence expressed. It inverts non-reversibly the sequence between the recognition sites in the PB state, which now are in the LR state. This leaves/positions the recombinase and RDF coding sequences in the same reading frame (reverse).
                                                    </p>
                                                </div>
                                            </div>

                                            <div class="col-sm-6 row wiki-card-6">
                                                <div class="col-sm-2 text-end">
                                                    <h1>
                                                        B
                                                    </h1>
                                                </div>
                                                <div class="col-sm-10">
                                                    <h2><b>Second generation</b></h2>
                                                    <p>
                                                        Because of the past inversion, the recombinase 1 and its respective RDF are reverse and the first terminator is now inactive, which allows the second recombinase to be expressed. The recombinase 2 inverts the sequence between the recognition sites in the PB state, which become the LR state. This leaves the recombinase 2 and RDF coding sequences in the same reading frame (reverse).
                                                    </p>
                                                </div>
                                            </div>
                                        </div>

                                        <div class="container wiki-card  mx-auto p-3 row wikicard-6-2">
                                            <div class="col-sm-6 row wiki-card-6 ">
                                                <div class="col-sm-2 text-end">
                                                    <h1>
                                                        C
                                                    </h1>
                                                </div>
                                                <div class="col-sm-10">
                                                    <h2><b>Third generation</b> </h2>
                                                    <p>
                                                        The recombinase 1, RDF and terminator are now in the correct reading strand. Only the rec 1 and the RDF are expressed, thus allowing the invertion between/of the sites in the LR state back into the PB state. This leaves the RDF 1 coding sequence in the same position (forward) and the recombinase 1 in the reverse.
                                                    </p>
                                                </div>
                                            </div>

                                            <div class="col-sm-6 row wiki-card-6">
                                                <div class="col-sm-2 text-end">
                                                    <h1>
                                                        D
                                                    </h1>
                                                </div>
                                                <div class="col-sm-10">
                                                    <h2><b>Fourth generation</b></h2>
                                                    <p>
                                                        Recombinase 1, and recombinase 2 with the RDF 2 are in the reverse strand and the first two promoters are inactive, thus, only the RDF of the recombinase 1 and the third recombinase are expressed. The RDF has no effect without its corresponding recombinase. The recombinase 3 inverts irreversibly the sequence between the recognition sites in the PB state, which now are in the LR state. This leaves the recombinase 3 and RDF 3 coding sequences in the same reading frame (reverse).
                                                    </p>
                                                </div>
                                            </div>
                                        </div>


                                        <div class="container wiki-card  mx-auto p-3 row wikicard-6-2">
                                            <div class="col-sm-6 row wiki-card-6 ">
                                                <div class="col-sm-2 text-end">
                                                    <h1>
                                                        E
                                                    </h1>
                                                </div>
                                                <div class="col-sm-10">
                                                    <h2><b>Fifth generation</b> </h2>
                                                    <p>
                                                        The RDF 2 and rec 1 are expressed. The RDF 2 has no effect without its corresponding recombinase. The recombinase 1 inverts irreversibly the sequence between the recognition sites in the PB state, which now are in the LR state. This leaves the recombinase 1 and RDF 1 coding sequences in the same reading frame (reverse). 
                                                    </p>
                                                </div>
                                            </div>

                                            <div class="col-sm-6 row wiki-card-6">
                                                <div class="col-sm-2 text-end">
                                                    <h1>
                                                        F
                                                    </h1>
                                                </div>
                                                <div class="col-sm-10">
                                                    <h2><b>Sixth generation</b></h2>
                                                    <p>
                                                        Only the rec 2 and the RDF 2 are expressed, thus allowing the invertion between/of the sites in the LR state back into the PB state. This leaves the RDF 2 coding sequence in the same position (forward) and the recombinase 2 in the reverse. 
                                                    </p>
                                                </div>
                                            </div>
                                        </div>


                                        <div class="container wiki-card  mx-auto p-3 row wikicard-6-2">
                                            <div class="col-sm-6 row wiki-card-6 ">
                                                <div class="col-sm-2 text-end">
                                                    <h1>
                                                        G
                                                    </h1>
                                                </div>
                                                <div class="col-sm-10">
                                                    <h2><b>Seventh generation</b> </h2>
                                                    <p>
                                                        The RDF of the recombinase 2 and the first recombinase with its corresponding RDF are expressed. The RDF 2 has no effect without its corresponding recombinase. The recombinase 1 with the RDF invert the sequence between the recognition sites in the LR state back into the PB state. This leaves the  RDF 1 coding sequence in the same position (forward) and the recombinase 1 in the reverse. 
                                                    </p>
                                                </div>
                                            </div>

                                            <div class="col-sm-6 row wiki-card-6">
                                                <div class="col-sm-2 text-end">
                                                    <h1>
                                                        H
                                                    </h1>
                                                </div>
                                                <div class="col-sm-10">
                                                    <h2><b>Eight generation</b></h2>
                                                    <p>
                                                        The fourth recombinase is expressed which inverts the complete circuit which is located between the PB sites and __ them in the  LR conformation.
                                                    </p>
                                                </div>
                                            </div>
                                        </div>



                                        <div class="container wiki-card  mx-auto p-3 row wikicard-6-2">
                                            <div class="col-sm-6 row wiki-card-6 ">
                                                <div class="col-sm-2 text-end">
                                                    <h1>
                                                        I
                                                    </h1>
                                                </div>
                                                <div class="col-sm-10">
                                                    <h2><b>Ninth generation</b> </h2>
                                                    <p>
                                                        The RDF of the recombinase 3 and the recombinase 1 are expressed step 1 is repeated. The RDF 3 has no effect without its corresponding recombinase. The recombinase 1 inverts non-reversibly the sequence between the PB recognition sites, which now are in the LR conformation. This leaves/positions the recombinase 1 and RDF 1 coding sequences in the same reading frame (reverse). 
                                                    </p>
                                                </div>
                                            </div>

                                            <div class="col-sm-6 row wiki-card-6">
                                                <div class="col-sm-2 text-end">
                                                    <h1>
                                                        J
                                                    </h1>
                                                </div>
                                                <div class="col-sm-10">
                                                    <h2> <b>Tenth generation</b></h2>
                                                    <p>
                                                        The RDF 3 and recombinase 2 are expressed. The RDF 3 has no effect without its corresponding recombinase. The recombinase 2 inverts non-reversibly the sequence between the PB recognition sites, which turn/ now are in the LR conformation. This leaves/positions the recombinase 2 and RDF 2 coding sequences in the same reading frame (reverse).
                                                    </p>
                                                </div>
                                            </div>
                                        </div>






                                    </div>
                                </div>
                            </div>



                            <div class="accordion-item">
                                <h2 class="accordion-header" id="panelsStayOpen-headingThree">
                                    <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
                                        data-bs-target="#panelsStayOpen-collapseThree" aria-expanded="false"
                                        aria-controls="panelsStayOpen-collapseThree">
                                        Expand Count
                                    </button>
                                </h2>
                                <div id="panelsStayOpen-collapseThree" class="accordion-collapse collapse"
                                    aria-labelledby="panelsStayOpen-headingThree">
                                    <div class="accordion-body">

                                        <div class="container  mx-auto p-3 wikicard-12-img">

                                            <div class="text-center">
                                                <img src="https://2021.igem.org/wiki/images/f/f9/T--TEC_COSTA_RICA--Description_GSC_16_GIFAmplify01.gif" alt=""
                                                    style="width:100% center" class="w-50">
                                            </div>
                                        </div>



                                        <div
                                            class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                                            <h2>Expand Count </h2>

                                            <p class="">
                                                A single segment is analyzed in this animation, it consists of an inducer for the inducible promoter that controls the fourth recombinase, a Cas9 and a ligase, the recombinases’ directionality factor which allows it to invert the sequence from LR to PB, and two single guides which enable the excision of the segment by the Cas9’s action. Hence, one by one the <b>auto-excisable segments</b> are expressed and deleted until the last one is reached, giving way for the last module to express itself. It’s important to note that only a slight change in terminator sequence is needed to yield dysfunctional activity for each module. 
                                            </p>
                                        </div>








                                    </div>
                                </div>
                            </div>


                            <div class="accordion-item">
                                <h2 class="accordion-header" id="panelsStayOpen-headingForth">
                                    <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
                                        data-bs-target="#panelsStayOpen-collapseForth" aria-expanded="false"
                                        aria-controls="panelsStayOpen-collapseForth">
                                        Kill Switch
                                    </button>
                                </h2>
                                <div id="panelsStayOpen-collapseForth" class="accordion-collapse collapse"
                                    aria-labelledby="panelsStayOpen-headingForth">
                                    <div class="accordion-body">






                                        <div
                                            class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                                            <h2>Kill Switch </h2>

                                            <p class="">
                                                When the RelE toxin is expressed, translation of proteins slowly comes to a halt. Simultaneously genomic DNA will be destroyed by the NucA toxin, the Gmk gene will be targeted by the Cas9 and our construct will be destroyed by this very same enzyme. The coupled action of these enzymes will ultimately <b>kill the cell without releasing its genetic contents</b>. 
                                            </p>
                                        </div>






                                    </div>
                                </div>
                            </div>



                        </div>


                    </div>



                    <!-- EXTRAS -->
                    <div v-if="current_slide_index['circuit'].index==5" class="container">





                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2>Overcount Prevention </h2>

                            <p class="">
                                “A potential challenge in designing a robust biological counter is the ability to count at completion of the event. The existing designs of the counters are sensitive to the pulse duration–a brief pulse will be ignored and a lengthy pulse can cause the counter to count ahead” (Noman et al., 2016). Given the concerns over the regulation of the count expressed by Dr. Pakpoom and Dr. Bonnet, further explained in engineering success, we designed a module to prevent overcount. This system works by placing an proteic inducer for an external promoter and an operator site after the main promoter. When active, the external promoter will produce a repressor which will in turn bind to the operator site preventing translation. The central concept to it is <b>affinity</b>, both the proteic inducer and the operator affinity are tunable, the first more than the latter. Once a pulse is received by the main promoter the proteic inducer will be expressed, along with the respective recombinase depending on the circuit’s state. If designed correctly, the affinity of the proteic inducer and the repressor will be just so that exactly the right amount of recombinase will be produced in order to change state without doing it twice or being shut off before being able to act. We designed a construct to prove this hypothesis, you can check our work on it in Wet Lab.
                            </p>
                        </div>
                        <div class="container  mx-auto p-3 wikicard-12-img">
                            <div class="text-center">
                                <img src="//placehold.co/1200x300?text=snippet" alt="" style="width:100% center"
                                    class="w-75">
                            </div>
                        </div>

                        <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                            <div class="col-sm-6">
                                <!-- <h2>title </h2> -->
                                <!-- <h3>subtitle </h3> -->
                                <p>
                                    Another possible solution to the response of a continuous pulse would be to include an <b>incoherent feedforward loop</b> to regulate the type of pulse the system responds to, as suggested by PhD. Rodrigo Mora. This type of standardized regulation consists of an activator which “regulates both a gene and a repressor of the gene”, similar to the manner in which we proposed our overcount system (Goentoro et al., 2009).
                                </p>
                            </div>


                        </div>

                        <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                            <div class="col-sm-6">
                                <h2>Promoters </h2>
                                <p>
                                    Additionally, we propose the use of inducible promoters (such as pBAD) instead of pnrd for <b>user-controlled counter systems</b>. These could even involve the association of the counter to specific environmental conditions such as pH, temperature, presence of certain molecules and others can easily be done by changing the initiating module to one dependent on these conditions. 
                                </p>
                            </div>

                            <div class="col-sm-6 justify-content-md-center">
                                <img class="img-square align-middle " src="https://2021.igem.org/wiki/images/b/b2/T--TEC_COSTA_RICA--Description_GSC_3_IconPromoter_Extras.png"
                                    all="">
                            </div>
                        </div>


                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2>Containment </h2>
                            <h3>Plasmid specific </h3>

                            <p class="">
                                A <b>toxin-antitoxin</b> could also be used for regulation and the actual kill (fourth device), in our proposal it’s counter effective to have another regulatory device of this magnitude but for systems in which a plasmid needs to be added and contained it could be useful, for example, the toxin could be placed on the plasmid and the respective antitoxin on the genome, avoiding horizontal gene transfer, because the plasmid will get the toxin but not the antitoxin to persist in other microorganisms (Torres et al., 2016). However, the toxin-antitoxin systems are usually complemented by a synthetic auxotrophy in which cells can only grow in the presence of an exogenously supplied metabolite that limits the cell survival under desired conditions, but implicates some risks (Chan et al., 2016). Toxin-antitoxin systems have been reported with high mobility between genomes through horizontal gene transfer and its stability in the genome is still uncertain due to mutations accumulation due to genetic drift (Van Melderen & Saavedra, 2009), and auxotrophy could be satisfied by metabolite cross-feeding or essential small molecules on environmental conditions (Gallagher et al., 2015), besides it requires extensive genome-wide engineering and reprogrammation for different environmental conditions is complicated (Chan et al., 2015). Nevertheless, we propose Gmk (https://www.uniprot.org/uniprot/P60546#sequences) as an auxotrophy gene, which has been reported as an essentialness by single-gene knocking out technique (Baba et al., 2006). It is important to denote that some toxin-antitoxin systems are able to lysate cells (Van Melderen & Saavedra, 2009), which is not beneficial for the optimal work timing of the whole system we propose. 
                            </p>
                        </div>

                        <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                            <div class="col-sm-6">
                                <!-- <h2>title </h2> -->
                                <h3>Specific toxins </h3>
                                <p>
                                     If a toxin could be designed to target our strain and only our strain of E. coli it could be released from the cells that first reach the end of the counter, <b>reducing the noise</b> of signal processing from each cell and ensuring that even cells with flaws in the system are killed.
                                </p>
                            </div>

                            <div class="col-sm-6 justify-content-md-center">
                                <img class="img-square align-middle " src="https://2021.igem.org/wiki/images/9/9a/T--TEC_COSTA_RICA--Description_GSC_4_IconContainment_Extras.png"
                                    all="">
                            </div>
                        </div>



                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2>Growth phase association: quorum sensing </h2>

                            <p class="">
                                Suggested by our users, i.e. M.Sc Alexander Schmidt, this proposal can be achieved in the same manner as the one described in the previous “promoters” section. Nonetheless, its implications are much more extensive, since “quorum sensing allows groups of bacteria to synchronously alter behaviour in response to changes in the population density and species composition of the vicinal community” (Mukherjee & Bassler, 2019). “QS is likely to control behaviours that are crucial for the development and success of these communities in diverse environments such as the human gut and chronic infections in humans” (Whiteley et al., 2017). According to (Wu et al., 2020) “<b>Dynamic control of bacterial populations</b> usually includes population size control, dynamic metabolic engineering for desirable products and the regulation of various physiological activities”. This regulation mechanism has been used in the development “of various genetic circuits such as genetic oscillators, toggle switches and logic gates with AHL-based QS devices in Gram-negative bacteria” (Wu et al., 2020).
                            </p>
                        </div>
                        <div class="container  mx-auto p-3 wikicard-12-img">

                            <div class="text-center">
                                <img src="https://2021.igem.org/wiki/images/5/52/T--TEC_COSTA_RICA--Description_GSC_5_IconQuoromSensing_Extras.png" alt="" style="width:100% center"
                                    class="w-75">
                            </div>
                        </div>








                    </div>


                    <!-- COMPARISON -->
                    <div v-if="current_slide_index['circuit'].index==6" class="container">
                        To further demonstrate the novelty and capacity of our counter circuit we present a comparison with previously developed counters, highlighting their characteristics of interest.

                    </div>

                    <!-- References -->
                    <div v-if="current_slide_index['circuit'].index==7" class="container">



                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2> </h2>
                            <h3> </h3>

                            <p class="">
                             Adan, A., Alizada, G., Kiraz, Y., Baran, Y., & Nalbant, A. (2017). Flow cytometry: basic principles and applications. Critical reviews in biotechnology, 37(2), 163-176. https://doi.org/10.3109/07388551.2015.1128876 <br><br>
Arkin, A. P., & Fletcher, D. A. (2006). Fast, cheap and somewhat in control. Genome biology, 7(8), 1-6. https://doi.org/10.1186/gb-2006-7-8-114 <br><br>
Baba, T., Ara, T., Hasegawa, M., Takai, Y., Okumura, Y., Baba, M., & Mori, H. (2006). Construction of Escherichia coli K‐12 in‐frame, single‐gene knockout mutants: the Keio collection. Molecular systems biology, 2(1), 2006-0008. https://doi.org/10.1038/msb4100050 <br><br>
Beal, J., Baldwin, G. S., Farny, N. G., Gershater, M., Haddock-Angelli, T., Buckley-Taylor, R., & iGEM Interlab Study Contributors. (2021). Comparative analysis of three studies measuring fluorescence from engineered bacterial genetic constructs. PloS one, 16(6), e0252263. https://doi.org/10.1371/journal.pone.0252263 <br><br>
Beal, J., Haddock-Angelli, T., Farny, N., & Rettberg, R. (2018). Time to get serious about measurement in synthetic biology. Trends in biotechnology, 36(9), 869-871. doi:10.1016/j.tibtech.2018.05.003 <br><br>
Bonnet, J., Subsoontorn, P., & Endy, D. (2012). Rewritable digital data storage in live cells via engineered control of recombination directionality. Proceedings of the National Academy of Sciences, 109(23), 8884-8889. https://doi.org/10.1073/pnas.1202344109 <br><br>
Canton, B., Labno, A., & Endy, D. (2008). Refinement and standardization of synthetic biological parts and devices. Nature biotechnology, 26(7), 787-793. DOI: 10.1038/nbt1413 <br><br>
Chan, C. T., Lee, J. W., Cameron, D. E., Bashor, C. J., & Collins, J. J. (2016). 'Deadman' and 'Passcode' microbial kill switches for bacterial containment. Nature chemical biology, 12(2), 82-86. doi: 10.1038/nchembio.1979 <br><br>
EcoCyc. [Data base]. Escherichia coli K-12 substr. MG1655 reference genome (EcoCyc). https://biocyc.org/gene?orgid=ECOLI&id=GUANYL-KIN-MONOMER# <br><br>
Englaender, J. A., Jones, J. A., Cress, B. F., Kuhlman, T. E., Linhardt, R. J., & Koffas, M. A. (2017). Effect of genomic integration location on heterologous protein expression and metabolic engineering in E. coli. ACS synthetic biology, 6(4), 710-720. doi:10.1021/acssynbio.6b00350 <br><br>
Fedorec, A. J., Robinson, C. M., Wen, K. Y., & Barnes, C. P. (2020). FlopR: an open source software package for calibration and normalization of plate reader and flow cytometry data. ACS synthetic biology, 9(9), 2258-2266. doi: 10.1021/acssynbio.0c00296 <br><br>
Fernandez-Rodriguez, J., Yang, L., Gorochowski, T. E., Gordon, D. B., & Voigt, C. A. (2015). Memory and combinatorial logic based on DNA inversions: dynamics and evolutionary stability. ACS synthetic biology, 4(12), 1361-1372. doi:10.1021/acssynbio.5b00170 <br><br>
Fredrickson, J. K., Bentjen, S. A., Bolton Jr, H., Li, S. W., Ligotke, M. W., McFadden, K. M., & Van Voris, P. (1989). Evaluation of terrestrial microcosms for assessing the fate and effects of genetically engineered microorganisms on ecological processes (No. PNL-6850). Pacific Northwest Lab., Richland, WA (USA). https://doi.org/10.2172/6294057 <br><br>
Friedland, A. E., Lu, T. K., Wang, X., Shi, D., Church, G., & Collins, J. J. (2009). Synthetic gene networks that count. Science, 324(5931), 1199-1202. doi: 10.1126/science.1172005 <br><br>
Gallagher, R. R., Patel, J. R., Interiano, A. L., Rovner, A. J., & Isaacs, F. J. (2015). Multilayered genetic safeguards limit growth of microorganisms to defined environments. Nucleic acids research, 43(3), 1945-1954. doi: 10.1093/nar/gku1378 <br><br>
Gil, R., Silva, F. J., Peretó, J., & Moya, A. (2004). Determination of the core of a minimal bacterial gene set. Microbiology and Molecular Biology Reviews, 68(3), 518-537. DOI: 10.1128/MMBR.68.3.518-537.2004 <br><br>
Goentoro, L., Shoval, O., Kirschner, M. W., & Alon, U. (2009). The incoherent feedforward loop can provide fold-change detection in gene regulation. Molecular cell, 36(5), 894-899. doi: 10.1016/j.molcel.2009.11.018 <br><br>
Guiziou, S., Mayonove, P., & Bonnet, J. (2019). Hierarchical composition of reliable recombinase logic devices. Nature communications, 10(1), 1-7. https://doi.org/10.1038/s41467-019-08391-y <br><br>
Heyde, K. C., & Ruder, W. C. (2015). Exploring host-microbiome interactions using an in silico model of biomimetic robots and engineered living cells. Scientific reports, 5(1), 1-12. https://doi.org/10.1038/srep11988 <br><br>
Howell, M., Daniel, J. J., & Brown, P. J. (2017). Live cell fluorescence microscopy to observe essential processes during microbial cell growth. Journal of visualized experiments: JoVE, (129). doi: 10.3791/56497 <br><br>
Hsiao, V., Hori, Y., Rothemund, P. W., & Murray, R. M. (2016). A population‐based temporal logic gate for timing and recording chemical events. Molecular systems biology, 12(5), 869. DOI: 10.15252/msb.20156663 <br><br>
Inamori, Y., Murakami, K., Sudo, R., Kurihara, Y., & Tanaka, N. (1992). Environmental assessment method for field release of genetically engineered microorganisms using microcosm systems. Water Science and Technology, 26(9-11), 2161-2164. https://doi.org/10.2166/wst.1992.0686 <br><br>
Jack, B. R., Leonard, S. P., Mishler, D. M., Renda, B. A., Leon, D., Suárez, G. A., & Barrick, J. E. (2015). Predicting the genetic stability of engineered DNA sequences with the EFM calculator. ACS synthetic biology, 4(8), 939-943. doi:10.1021/acssynbio.5b00068 <br><br>
Jin, E., Wong, L., Jiao, Y., Engel, J., Holdridge, B., & Xu, P. (2017). Rapid evolution of regulatory element libraries for tunable transcriptional and translational control of gene expression. Synthetic and systems biotechnology, 2(4), 295-301. doi: 10.1016/j.synbio.2017.10.003 <br><br>
Kim, S., Jeong, H., Kim, E. Y., Kim, J. F., Lee, S. Y., & Yoon, S. H. (2017). Genomic and transcriptomic landscape of Escherichia coli BL21 (DE3). Nucleic acids research, 45(9), 5285-5293. https://doi.org/10.1093/nar/gkx228 <br><br>
Lewis, M. (2011). A tale of two repressors. Journal of molecular biology, 409(1), 14-27. doi: 10.1016/j.jmb.2011.02.023 <br><br>
Liu, G., Zhang, Y., & Zhang, T. (2020). Computational approaches for effective CRISPR guide RNA design and evaluation. Computational and structural biotechnology journal, 18, 35-44. https://doi.org/10.1016/j.csbj.2019.11.006 <br><br>
de Lorenzo, V. (2009). Recombinant bacteria for environmental release: what went wrong and what we have learnt from it. Clinical Microbiology and Infection, 15, 63-65. https://doi.org/10.1111/j.1469-0691.2008.02683.x <br><br>
McGinness, K. E., Baker, T. A., & Sauer, R. T. (2006). Engineering controllable protein degradation. Molecular cell, 22(5), 701-707. https://doi.org/10.1016/j.molcel.2006.04.027 <br><br>
McKinnon, K. M. (2018). Flow cytometry: an overview. Current protocols in immunology, 120(1), 5-1.  https://doi.org/10.1002/cpim.40 <br><br>
MedlinePlus. (s.f.) What are genome editing and CRISPR-Cas9? https://medlineplus.gov/genetics/understanding/genomicresearch/genomeediting/ <br><br>
Merrick, C. A., Zhao, J., & Rosser, S. J. (2018). Serine integrases: advancing synthetic biology. ACS synthetic biology, 7(2), 299-310. DOI: 10.1021/acssynbio.7b00308 <br><br>
Mukherjee, S., & Bassler, B. L. (2019). Bacterial quorum sensing in complex and dynamically changing environments. Nature Reviews Microbiology, 17(6), 371-382. https://doi.org/10.1038/s41579-019-0186-5 <br><br>
Noman, N., Inniss, M., Iba, H., & Way, J. C. (2016). Pulse detecting genetic circuit–a new design approach. PLoS One, 11(12), e0167162. https://doi.org/10.1371/journal.pone.0167162 <br><br>
Pasotti, L., Zucca, S., Lupotto, M., De Angelis, M. G. C., & Magni, P. (2011). Characterization of a synthetic bacterial self-destruction device for programmed cell death and for recombinant proteins release. Journal of biological engineering, 5(1), 1-12. https://doi.org/10.1186/1754-1611-5-8 <br><br>
Perez-Garcia, O., Lear, G., & Singhal, N. (2016). Metabolic network modeling of microbial interactions in natural and engineered environmental systems. Frontiers in microbiology, 7, 673. https://doi.org/10.3389/fmicb.2016.00673 <br><br>
Politz, M. C., Copeland, M. F., & Pfleger, B. F. (2013). Artificial repressors for controlling gene expression in bacteria. Chemical communications, 49(39), 4325-4327. doi: 10.1039/c2cc37107c <br><br>
Roquet, N., Soleimany, A. P., Ferris, A. C., Aaronson, S., & Lu, T. K. (2016). Synthetic recombinase-based state machines in living cells. Science, 353(6297).  DOI: 10.1126/science.aad8559 <br><br>
Sleight, S. C., Bartley, B. A., Lieviant, J. A., & Sauro, H. M. (2010). Designing and engineering evolutionary robust genetic circuits. Journal of biological engineering, 4(1), 1-20. https://doi.org/10.1186/1754-1611-4-12 <br><br>
Stark, W. M. (2017). Making serine integrases work for us. Current opinion in microbiology, 38, 130-136. https://doi.org/10.1016/j.mib.2017.04.006 <br><br>
Stirling, F., Bitzan, L., O’Keefe, S., Redfield, E., Oliver, J. W., Way, J., & Silver, P. A. (2017). Rational design of evolutionarily stable microbial kill switches. Molecular cell, 68(4), 686-697. https://doi.org/10.1016/j.molcel.2017.10.033 <br><br>
Su, T., Liu, F., Gu, P., Jin, H., Chang, Y., Wang, Q., & Qi, Q. (2016). A CRISPR-Cas9 assisted non-homologous end-joining strategy for one-step engineering of bacterial genome. Scientific reports, 6(1), 1-11. https://doi.org/10.1038/srep37895 <br><br>
Su, T., Liu, F., Chang, Y., Guo, Q., Wang, J., Wang, Q., & Qi, Q. (2019). The phage T4 DNA ligase mediates bacterial chromosome DSBs repair as single component non-homologous end joining. Synthetic and systems biotechnology, 4(2), 107-112. doi: 10.1016/j.synbio.2019.04.001 <br><br>
Széliová, D., Krahulec, J., Šafránek, M., Lišková, V., & Turňa, J. (2016). Modulation of heterologous expression from PBAD promoter in Escherichia coli production strains. Journal of biotechnology, 236, 1-9. https://doi.org/10.1016/j.jbiotec.2016.08.004 <br><br>
Team: Cambridge. (2009). E.chromi. The International Genetically Engineered Machine iGEM. https://2009.igem.org/Team:Cambridge/Project/Amplification/Characterisation <br><br>
Team: Groningen. (2011). Count coli. The International Genetically Engineered Machine iGEM. http://2011.igem.org/Team:Groningen/project_description <br><br>
Team: Paris Liliane Bettencourt. (2010). Memo-cell. The International Genetically Engineered Machine iGEM. http://2010.igem.org/Team:Paris_Liliane_Bettencourt/Project/Memo-cell <br><br>
Team: SJTU-BioX-Shanghai. (2009). E.coli the napper.  he International Genetically Engineered Machine iGEM. https://2009.igem.org/Team:SJTU-BioX-Shanghai/Project_design#Overview <br><br>
Team: Stanford-Brown. (2012). The transit of synthetic astrobiology. The International Genetically Engineered Machine iGEM. http://2012.igem.org/Team:Stanford-Brown <br><br>
Team: SYSU China. (2015). Micro-time system. The International Genetically Engineered Machine iGEM. http://2015.igem.org/Team:SYSU_CHINA/Design <br><br>
Team: UT-Tokyo. (2016). Like does not beget like. The way to make a 100-stage cycle. The International Genetically Engineered Machine iGEM. http://2016.igem.org/Team:UT-Tokyo/Project#The_way_to_make_a_100-stage_cycle <br><br>
The International Genetically Engineered Machine iGEM. (s.f.a). Promoters/Catalog. https://parts.igem.org/Promoters/Catalog <br><br>
The International Genetically Engineered Machine iGEM. (s.f.b). Terminators. https://parts.igem.org/Terminators. <br><br>
Tomimatsu, K., Kokura, K., Nishida, T., Yoshimura, Y., Kazuki, Y., Narita, M., & Ohbayashi, T. (2017). Multiple expression cassette exchange via TP 901‐1, R4, and Bxb1 integrase systems on a mouse artificial chromosome. FEBS open bio, 7(3), 306-317.  https://doi.org/10.1002/2211-5463.12169 <br><br>
Torres, L., Krüger, A., Csibra, E., Gianni, E., & Pinheiro, V. B. (2016). Synthetic biology approaches to biological containment: pre-emptively tackling potential risks. Essays in Biochemistry, 60(4), 393-410. doi: 10.1042/EBC20160013 <br><br>
Van Melderen, L., & Saavedra De Bast, M. (2009). Bacterial toxin–antitoxin systems: more than selfish entities?. PLoS genetics, 5(3), e1000437. https://doi.org/10.1371/journal.pgen.1000437 <br><br>
Wang, X., Tang, B., Ye, Y., Mao, Y., Lei, X., Zhao, G., & Ding, X. (2017). Bxb1 integrase serves as a highly efficient DNA recombinase in rapid metabolite pathway assembly. Acta biochimica et biophysica Sinica, 49(1), 44-50. https://doi.org/10.1093/abbs/gmw115 <br><br>
Whiteley, M., Diggle, S. P., & Greenberg, E. P. (2017). Progress in and promise of bacterial quorum sensing research. Nature, 551(7680), 313-320. ​​doi:10.1038/nature24624 <br><br>
Wu, S., Liu, J., Liu, C., Yang, A., & Qiao, J. (2020). Quorum sensing for population-level control of bacteria and potential therapeutic applications. Cellular and Molecular Life Sciences, 77(7), 1319-1343. https://doi.org/10.1007/s00018-019-03326-8 <br><br>
Xu, Z., & Brown, W. R. (2016). Comparison and optimization of ten phage encoded serine integrases for genome engineering in Saccharomyces cerevisiae. BMC biotechnology, 16(1), 1-10. https://doi.org/10.1186/s12896-016-0241-5 <br><br>
Xu, X., & Qi, L. S. (2019). A CRISPR–dCas toolbox for genetic engineering and synthetic biology. Journal of molecular biology, 431(1), 34-47. https://doi.org/10.1016/j.jmb.2018.06.037 <br><br>
Yang, L., Nielsen, A. A., Fernandez-Rodriguez, J., McClune, C. J., Laub, M. T., Lu, T. K., & Voigt, C. A. (2014). Permanent genetic memory with> 1-byte capacity. Nature methods, 11(12), 1261-1266. DOI: 10.1038/nmeth.3147 <br><br>
Zhao, J., Pokhilko, A., Ebenhöh, O., Rosser, S. J., & Colloms, S. D. (2019). A single-input binary counting module based on serine integrase site-specific recombination. Nucleic acids research, 47(9), 4896-4909. doi: 10.1093/nar/gkz245 <br><br>
                            </p>


                        </div>



                    </div>

                </div>











                <!-- pills-circuit -->


                <!-- pills-framework -->
                <div class="tab-pane fade" id="pills-framework" role="tabpanel" aria-labelledby="pills-framework-tab">


                    <main-carousel v-bind:current_slide="current_slide_index['framework']"
                        v-bind:slides_content="slides_content['framework']" :id="1">
                    </main-carousel>


                    <!-- intro  -->
                    <div v-if="current_slide_index['framework'].index==0" class="container">

                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2>Our motivation </h2>
                            <h3> </h3>

                            <p class="">
                                When we decided to explore sequential logic in biology through our {software}, we encountered two determinant situations from which this conceptual framework was born. The first one was, after several rounds of interviews, <b>most experts we talked to did not know about the relationship between sequential logic and biology</b>. Second, as this is a rather unexplored approach, there is <b>minimal literature and guidance</b> on how to assess a sequentially dependent biological system or how to conceptualize a circuit as a state machine. Due to the lack of information on the topic, we first had to discuss extensively and define foundational information in order to establish the right approach for the software's design. Therefore, we chose to compile what we had gathered and expand on it to generate a <b>guide</b> that would help not only us as the <b>developers</b> of the software, but also its immediate <b>users</b> and even <b>those who wish to implement another tool</b> or just <b>better understand</b> sequential logic in biology.<br>
                            </p>
                        </div>

                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2>Our inspiration </h2>
                            <h3> </h3>

                            <p class="">
                                Even though the application of sequential logic in synthetic biology is rather unexplored, the area has a lot of potential. As Roquet <i>et al.</i>(2016) remarked, <b>despite functional state machines' prospect to be transformative tools in the understanding and engineering of complex biological systems, their implementation has been hindered because of the absence of a framework that would offer a scalable and generalizable approach</b>. The proposal Roquet <i>et al</i>. (2016) developed was a recombinase-based framework for the implementation of state machines in cells, by encoding the state in a particular DNA sequence; they expanded on it with a large, searchable database of such state machines registers to help researchers design circuits. Even previously, Oishi and Klavins (2014) proposed a framework for building finite state machines with engineered regulatory networks based on repressing transcription factors. However, these approaches pursue highly specific methods and do not focus on a conceptual guide for a wide range of functionalities. Therefore, a more <b>general and modular toolset</b> for the exploration of sequential logic based synthetic biology would greatly benefit <b>scientists and professionals with different levels of knowledge.</b> <br>
                            </p>
                        </div>

                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2>Our goal </h2>
                            <h3> </h3>

                            <p class="">
                                We set as our objective the proposal of a framework composed of a set of rules and concepts for this approach, which would be the <b>base for the exploration of an incipient field</b> and for the <b>development of novel tools</b>. Such a guide would serve as a structure upon which to design, construct, model and analyze -either computationally or manually- synthetic gene circuits from a sequential logic standpoint. Both the purpose for this framework and the process by which we achieved it, was to ask the right questions that represented our inquiries and offer the corresponding answers to help and guide others. Such questions and answers would begin a discussion to further advance and understand this area. Beyond the applications that can be given to this framework and the work it can support, it also offers a <b>comprehensive and paradigm-shifting guide</b> in itself; therefore anyone who comes across it can dive into the analysis and advantages of biological systems as sequential systems, providing them with a new perspective.<br>
                            </p>
                        </div>

                    </div>

                    <!-- Description -->
                    <div v-if="current_slide_index['framework'].index==1" class="container">

                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <!-- <h2>title </h2> -->
                            <!-- <h3>subtitle </h3> -->

                            <p class="">
                                Here we propose a conceptual framework, which is an <b>efficient and modular approach on how to tackle sequentially dependent biological tasks</b>, specifically for synthetic genetic circuits and their design. It is a <b>proposal on how to describe in a high abstraction level the biological elements</b> involved in these types of sequential applications, such as parts, circuits, their environment and their interactions, <b>and their assembly to simulate a behavior as a finite state machine.</b><br>
                            </p>
                        </div>


                        <div class="container wiki-card wikicard-12 mx-auto p-3 row ">
                            <div class="col-sm-1 text-end">
                                <h1> 01 </h1>
                            </div>
                            <div class="col-sm-9">
                                <h2>Objective </h2>
                                <h3> </h3>

                                <p>
                                    The objective of this framework is to conceptualize, in an abstract and modular manner, <b>biological processes as state machines</b>; specifically synthetic genetic circuits for the exploration of their structure and behavior from a sequential logic point of view.<br>
                                </p>
                            </div>
                        </div>

                        <div class="container wiki-card wikicard-12 mx-auto p-3 row ">
                            <div class="col-sm-1 text-end">
                                <h1> 02 </h1>
                            </div>
                            <div class="col-sm-9">
                                <h2>Foundational Concepts </h2>
                                <h3> </h3>

                                <p>
                                    Even though we expound the {characteristics} of the biological problems that match with this approach, it is important to note from the beginning that this is a <b>proposal for complex, time and memory-dependent, or regulatory intricate systems</b>. Therefore, other circuits and systems exist whose behavior doesn't require to be abstracted as sequential logic. For more specific examples of applications, go to {implementation} where we expand on the uses we and the potential users we talked to propose.<br>
                                </p>
                            </div>
                        </div>

                        <div class="container wiki-card wikicard-12 mx-auto p-3 row ">
                            <div class="col-sm-1 text-end">
                                <h1> 03 </h1>
                            </div>
                            <div class="col-sm-9">
                                <h2>Scope and Applications </h2>
                                <h3> </h3>

                                <p>
                                    To better understand our approach, it is important to know that the framework is supported by three key concepts: <b>sequential logic, abstraction and modularity</b>. These foundational concepts are core to our project as they guide the development of the framework and are aligned with our purpose.<br>
                                </p>
                            </div>
                        </div>


                    </div>


                    <!-- Foundational Concept -->
                    <div v-if="current_slide_index['framework'].index==2" class="container">



                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2>Logic </h2>
                            <h3> </h3>

                            <p class="">
                                The conceptualization of a genetic circuit's function as sequential logic can be daunting and hard to replicate consistently. There are a number of elements that must be considered to properly represent the circuit's structure and behavior. In a sequential system, the <b>output depends not only on the present inputs but also on the current state</b>. For <b>synthetic biology</b>, a circuit and its environment can be portrayed as a sequential system or state machine when the <b>behavior can be described by states whose transitions are controlled by time-ordered inputs and are affected by past behavior and inputs</b>. The modification and behavior of circuits can be explored through the sequential progress of states. Exploring the potential states of a circuit requires recurrent simulations of its changes and behavior; hence, there must exist an iterable ready approach to tackle these types of problems. A biological process, natural or engineered, that has characteristics such as: being time-dependent, having highly interconnected regulatory networks, behaving differently with repeating inputs, requiring information to be saved stably (Letsou & Cai, 2016; Madec, Rosati, & Lallement, 2021; Magdevska <i>et al</i>., 2017; Oishi and Klavins, 2014; Roquet <i>et al.</i>, 2016), would greatly benefit from being conceptualized and analyzed as a sequential logic problem. Moreover, the ability to sequentially modulate the system's response offers high scalability as it enables larger or more complex applications (Zúñiga <i>et al</i>., 2020).<br>
                            </p>
                        </div>

                        <div class="container  mx-auto p-3 wikicard-12-img">

                            <div class="text-center">
                                <img src="https://2021.igem.org/wiki/images/f/f4/T--TEC_COSTA_RICA--Description_Framework_01_LogicFoundationalConcept.png" alt="" style="width:100% center"
                                    class="w-50">
                            </div>
                        </div>

                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2>Abstraction </h2>
                            <h3> </h3>

                            <p class="">
                                To perform a thorough exploration of the design space (<i>i.e.</i> possible architectures), the framework follows a highly abstract approach. This is necessary since there is a direct relationship between biological accuracy and modeling complexity; therefore, by abstracting most of the biological details we are able to <b>focus on performance, simplicity, modularity and expandability</b>. This tradeoff is valuable from a design perspective where it is more efficient to perform a <b>coarse analysis unlimited and unbiased</b> by biological technicalities and then focus on fine-tuning an oriented and pre-processed set of circuits, based on a specific use-case. This framework works in an area of what is biologically viable, but more complex biological challenges present when looking into implementing a circuit, such as transcription rates or residual protein concentration, are meant to be addressed at other following stages and solved through other existing solutions or tools. This approach allows for a general-purpose framework.<br>
                            </p>
                        </div>

                        <div class="container  mx-auto p-3 wikicard-12-img">

                            <div class="text-center">
                                <img src="https://2021.igem.org/wiki/images/4/43/T--TEC_COSTA_RICA--Description_Framework_02_AbstractionFoundationalConcept.png" alt="" style="width:100% center"
                                    class="w-50">
                            </div>
                        </div>

                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2>Modularity </h2>
                            <h3> </h3>

                            <p class="">
                                Every biological system is different and should not be represented by a static model or relation. Conceptualizations and models such as truth tables work with the coupling of multiple modules but focus on small scale inputs and outputs rather than an overall behaviour and functionality. When we build a device we focus on processes rather than specific responses, which is why we propose an {execution} through atomic functions that can be <b>stacked to achieve more complex behavior while maintaining simplicity and modularity</b>. The abstract representation we propose uses a modular approach in terms of composition, allowing the user to <b>adjust, append or remove concepts</b> according to the application. Everything is <b>built upon a primary building block</b> to enable modularity and scalability. <br>
                            </p>
                        </div>

                        <div class="container  mx-auto p-3 wikicard-12-img">

                            <div class="text-center">
                                <img src="https://2021.igem.org/wiki/images/4/4f/T--TEC_COSTA_RICA--Description_Framework_03_ModularityFoundationalConcept.png" alt="" style="width:100% center"
                                    class="w-50">
                            </div>
                        </div>






                    </div>

                    <!-- Framework´s architecture -->
                    <div v-if="current_slide_index['framework'].index==3" class="container">


                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">

                            <p class="">
                                The framework is divided into three stages to address all relevant aspects. First, the framework expands on how a circuit, cell or gene regulatory network can be a sequential system in the first phase: {biological state machines}, which dives into sequential logic applied in synthetic biology and its characteristics. Then, it defines the concepts involved in the rest of the framework in the {description phase}: an approach to conceptualize biological elements and their interactions. Lastly, the framework describes how to assess and develop circuits as state machines; the {execution stage} consists of the coupling of the first two, to fully describe and simulate, from a sequential logic standpoint, biological processes and regulatory networks from synthetic circuits.<br>
                            </p>
                        </div>

                        <div class="container  mx-auto p-3 wikicard-12-img">

                            <div class="text-center">
                                <img src="https://2021.igem.org/wiki/images/b/b2/T--TEC_COSTA_RICA--Description_Framework_15_DiagramArchitecture.png" alt="" style="width:100% center"
                                    class="w-75">
                            </div>
                        </div>



                    </div>

                    <!-- Biological state machines: -->
                    <div v-if="current_slide_index['framework'].index==4" class="container">


                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2>When? </h2>
                            <h3> </h3>

                            <p class="">
                                In order to conceptualize a circuit as a state machine, the <b>behavior must be dependent on the historical and present inputs</b>.<br>
                            </p>
                        </div>


                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2>How? </h2>
                            <h3> </h3>

                            <p class="">
                                The system’s <b>input</b> is the <b>composition of the circuit's environment.</b> Meanwhile, the <b>output</b> is an <b>expression profile</b> related to the state.
<br><br>


A <b>state</b>, is defined by two elements: the <b>circuit architecture</b>, <i>i.e.</i> the disposition of genetic parts or pair bases, and the <b>gene expression profile</b>. These elements were inspired by previous literature (Fritz <i>et al</i>., 2007; Oishi & Klavins, 2014; Roquet <i>et al</i>., 2016), as we considered multiple perspectives required and advantageous. Therefore, a change in state would be caused by a change in the DNA sequence, by a change in a gene's expression, or by a change in both. The first element, the circuit's architecture, corresponds to the order, orientation and sequence of parts; thus DNA modification caused by insertion, deletion, substitution, inversion, mutation, would cause a change in state. For this, enzymes that modify the DNA strand are essential. This is a highly evolutionary stable way to store the state as it is coded in the DNA and does not require a constant gene expression (with the corresponding metabolic burden) (Roquet <i>et al</i>., 2016). On the other hand, the expression profile refers to determined transcribed genes and their expression levels, which can be reflected in the environment. By extension, it considers gene regulatory networks and epigenetic mechanisms that affect the functionality of the circuit. Although a state defined by this aspect is not as stable, the objective is focused on considering relevant alterations as state changes, since they will affect the response to future inputs. If a change in expression causes a change in architecture or the other way around: if a change in architecture causes a change in expression, both could be grouped into the same state change.
<br><br>


To exemplify, we can consider the following system. <br>

A circuit with two different inducible promoters that achieves five states depending on the order of inputs. 
State A is the initial state without any induction. 
<br><br>


If inducer 1 is the first input, the promoter 1 is activated, the rfp and recombinase 1 are expressed. The rec inverts the sequence between the sites, which changes the circuit architecture and generates state B. 
<br><br>


If inducer 1 is added again, the rfp and rec 1 are expressed again but there is no relevant change and therefore there is no change of state.
If inducer 2 follows instead, the rec 2 is expressed and the sequence between the sites is excised, which generates state C. <br><br>


With input 1 next, the gfp is expressed, but if input 2 is inserted instead, the rec 2 is expressed but has no effect.
<br><br>


If instead of this order of inputs, the system in state A starts with inducer 2, the rec 2 is expressed, which excises the sequence between the sites generating state D. 
<br><br>

In state D, if the inducer 2 is input, the recombinase 2 is expressed but nothing happens. 
Instead, if inducer 1 is added next, the yfp and repressor are expressed. The repressor associates to the operator site, which modifies the expression profile, generating state E. 
<br><br>


In state E, with time as an input, the repressor degrades, which deactivates the operator site, going back to state D.
<br>

                            </p>
                        </div>

                        <div class="container  mx-auto p-3 wikicard-12-img">
                            <div class="text-center">
                                <img src="//placehold.co/1200x600?text=ANIMACION" alt="" style="width:100% center"
                                    class="w-75">
                            </div>
                        </div>

                        <div class="container  mx-auto p-3 wikicard-12-img">
                            <div class="text-center">
                                <img src="//placehold.co/1200x600?text=ANIMACION" alt="" style="width:100% center"
                                    class="w-75">
                            </div>
                        </div>


                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2>Characteristics </h2>
                            <h3> </h3>

                            <p class="">
                                The application of sequential logic to biological systems has certain characteristics which help understand better its applicability.<br>
                            </p>
                        </div>


                        <div class="container wiki-card wikicard-12 mx-auto p-3 row ">
                            <div class="col-sm-1 text-end">
                                <h1> 01 </h1>
                            </div>
                            <div class="col-sm-9">
                                <h2>Traceability </h2>
                                <h3> </h3>

                                <p>
                                    Sequential logic and state machines generate highly traceable systems since present actions respond to the previous history. In addition, it saves the <b>complete behavior of the circuit</b>, including the progression of states, inputs and outputs, <b>instead of only a DNA sequence</b>. Even further, according to the structure the state machine follows, the history of states can be known with only a single state. <br>
                                </p>
                            </div>
                        </div>

                        <div class="container wiki-card wikicard-12 mx-auto p-3 row ">
                            <div class="col-sm-1 text-end">
                                <h1> 02 </h1>
                            </div>
                            <div class="col-sm-9">
                                <h2>Memory-based </h2>
                                <h3> </h3>

                                <p>
                                    Since the current state of the system affects the output and future states, such <b>states can be used for saving information</b> even when the input is gone, rather than losing the response to the input. Therefore, such systems have a memory and can save data.<br>
                                </p>
                            </div>
                        </div>

                        <div class="container wiki-card wikicard-12 mx-auto p-3 row ">
                            <div class="col-sm-1 text-end">
                                <h1> 03 </h1>
                            </div>
                            <div class="col-sm-9">
                                <h2>Time-ordered </h2>
                                <h3> </h3>

                                <p>
                                    Since a sequential system responds differently to the same input depending on the current state, the specific point at which the input is present is determinant (Roquet <i>et al</i>., 2016). Due to this ability to <b>modulate the response according to a time-orderly induction</b>, state machines offer the capacity to <b>carry out complex functions</b> (Zúñiga <i>et al</i>., 2020) and <b>sequentially regulate targets</b> which, if using combinatorial logic, would get co-activated (Letsou & Cai, 2016). This enables applications such as counting the number of stimuli (such as in {our {counter}), or modifying the behavior depending on the order of inputs (Madec, Rosati, & Lallement, 2021). The latter is why cells accomplish a vast number of events and processes with limited genes (Letsou & Cai, 2016). State changes enable the consideration of <b>changes of a circuit over time and how that system evolves</b>, especially the environment.<br>
                                </p>
                            </div>
                        </div>

                        <div class="container wiki-card wikicard-12 mx-auto p-3 row ">
                            <div class="col-sm-1 text-end">
                                <h1> 04 </h1>
                            </div>
                            <div class="col-sm-9">
                                <h2>Alternative scenarios </h2>
                                <h3> </h3>

                                <p>
                                    Inherently, sequential logic is well fitted for the consideration of alternative routes as they can be naturally integrated as multiple states that can potentially follow the previous one. This is beneficial when dealing with <b>multiple alternative scenarios, exploring unknown routes</b>, and considering <b>error routes</b> and <b>potential mutations</b>. This allows the proposal of control plans, and a better and more thorough exploration, especially when the inputs or their order is not completely straightforward, as in biology. This can be coupled to <b>probabilities for each state change</b>, such as in a Markov Model. For example, a state machine could be built for a circuit in state 1 that after an input moves into desired state 2 but has a lesser probability of changing into state 3, the alternative scenario. Overall, this characteristic suits biology due to the stochastic element in biochemical reactions (Stelling, Szallasi, & Periwal; 2019).<br>
                                </p>
                            </div>
                        </div>

                        <div class="container wiki-card wikicard-12 mx-auto p-3 row ">
                            <div class="col-sm-1 text-end">
                                <h1> 05 </h1>
                            </div>
                            <div class="col-sm-9">
                                <h2>Multipurpose system </h2>
                                <h3> </h3>

                                <p>
                                    The use of logic in biology enables complex response patterns and overall processes. Since a <b>single input can lead to a different output</b> depending on the state, state machines facilitate multipurpose systems, also tightly related to alternative scenarios and a timed order response. With sequentiality, <b>from a single circuit</b> or state machine, <b>a great number of responses and fates can be derived</b>, as it naturally occurs in living cells.<br>
                                </p>
                            </div>
                        </div>

                        <div class="container wiki-card wikicard-12 mx-auto p-3 row ">
                            <div class="col-sm-1 text-end">
                                <h1> 06 </h1>
                            </div>
                            <div class="col-sm-9">
                                <h2>Easier troubleshooting </h2>
                                <h3> </h3>

                                <p>
                                    A state machine facilitates an emphasis on the <b>states' progression</b> rather than on a single final output, contrary to a combinatorial system composed of a truth table and logic gates whose structure is highly focused on the output as a 0/1 (on/off) response. Therefore, troubleshooting becomes straightforward when using sequential logic because <b>it doesn't require the circuit to reach its final state for it to fail</b>. In addition to this, by recording the circuit's history, the <b>specific input that went wrong</b> can be easily distinguished from the rest. This is possible given the system's <b>traceability</b> -due to the relation between the state and the output- together with the <b>capacity to measure the current state</b>, especially when the state is saved in the DNA sequence and can be known by sequencing it. In this case, the last state before the system failed can be known by looking at the sequence, instead of only having a failed/succeeded outcome without a way to know when and where the problem occurred.<br>
                                </p>
                            </div>
                        </div>

                        <div class="container wiki-card wikicard-12 mx-auto p-3 row ">
                            <div class="col-sm-1 text-end">
                                <h1> 07 </h1>
                            </div>
                            <div class="col-sm-9">
                                <h2>Concept extrapolation </h2>
                                <h3> </h3>

                                <p>
                                    Sequential logic can be explained as <b>decision making based on previous situations</b>. This is a concept analogous to human behavior: making decisions based on past experiences and current situations and opportunities. Furthermore, as a logic circuit, it is based on electronics. Therefore, the concept is easily comprehensible across, even beyond, biological sciences.<br>
                                </p>
                            </div>
                        </div>

                        <div class="container wiki-card wikicard-12 mx-auto p-3 row ">
                            <div class="col-sm-1 text-end">
                                <h1> 08 </h1>
                            </div>
                            <div class="col-sm-9">
                                <h2>Biologically suitable </h2>
                                <h3> </h3>

                                <p>
                                    Sequential logic is an approach distinctly similar to nature, as observed in highly interconnected gene regulatory networks, where regulators affect multiple targets and the targets are affected by multiple regulators (Letsou & Cai, 2016). These systems are not limited by a specific input-output response nor by the number of inducers, but rather respond to the temporally ordered presence of inducers (Letsou & Cai, 2016; Madec, Rosati, & Lallement, 2021; Roquet <i>et al</i>., 2016). By following sequential logic, the design of circuits <b>emulates and expands on the advantages nature offers</b>. This allows encompassing more <b>flexible</b> systems and proposing <b>highly scalable</b> designs  (Roquet <i>et al</i>., 2016).<br>

Furthermore, sequentiality <b>reduces the necessity of orthogonal inducers</b>, a relevant limitation in synthetic biology, as an induction directly linked to a point in time (or state) practically eliminates the possibility of <b>crosstalk</b>. Therefore, the number of inducers can be minimized if a system does not demand a different one for each desired output. This facilitates the design of circuits by excluding the biological limitation of quantity of highly specific inducers.<br>

Also related to the inducers is the <b>metabolic burden</b>, which can be reduced by storing information in the DNA sequence rather than requiring a constant gene expression or complex inducer combinations.<br>

Another limitation that is minimized is the <b>size of the circuit</b> since the number of pair bases can be reduced by modifying the DNA sequence rather than requiring multiple linearly positioned logic gates. This is especially important in synthetic circuits that must conform to a specific number of base pairs. <br>

Lastly, saving the state in the sequence itself leads to an increase in <b>evolutionary stability</b> (Zúñiga <i>et al</i>., 2020). As we mentioned previously, mutations and stochasticity are core elements of biology. By increasing the stability and offering an approach to consider alternative scenarios, state machines are perfectly designed to describe biological genetic systems. 

                                </p>
                            </div>
                        </div>


                    </div>

                    <!-- Description -->
                    <div v-if="current_slide_index['framework'].index==5" class="container">



                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2>Structural </h2>
                            <p class="">
                                Biological elements can have multiple representations, for example, SBOL is a standard for the representation of biological designs, especifically for the electronic exchange of information on their structural and functional aspects (Synthetic biology open language). Electrical engineering iconography is used for biological systems to illustrate logic gates as a function of inputs (Densmore & Anderson, 2009), and even further, strings of the letters ATCG are used to represent DNA sequences, common in DNA manipulation and design tools. Each representation best suits the purpose of its application, whether that is design, mathematical modeling, detailed description, synthesis, bioinformatics analysis, databases. For our framework, the structural segment encompasses the basic representations, as conceptual units, of biological elements that best suit our goal and seek to be abstract and modular to be used for further applications.<br>
                            </p>

                            <h3>Parts </h3>

                            <p class="">
                                A part is the framework's <b>abstraction of a coding or non-coding DNA sequence with a function</b>. Each part is a conceptual unit with a biological role, <i>e.g.</i> coding, recognizing, expressing, promoting/repressing transcribing. This abstraction is based on the function rather than the sequence itself, thus the biological role is the essential element of a part, regardless of the molecular structure it is directly related to: DNA, RNA or protein. Besides the biological role, parts can represent more specific information related to them, such as the corresponding DNA sequence. Parts have <b>properties, interact</b> with other parts or elements, and execute <b>functions</b>; and they are the <b>building blocks</b> for genetic circuits.<br>
                            </p>
                        </div>

                        <div class="container  mx-auto p-3 wikicard-12-img">

                            <div class="text-center">
                                <img src="https://2021.igem.org/wiki/images/e/e4/T--TEC_COSTA_RICA--Description_Framework_06_PartsDescription.png" alt="" style="width:100% center"
                                    class="w-75">
                            </div>
                        </div>


                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h3>Circuit </h3>

                            <p class="">
                                The circuit abstraction is what provides cohesion to a set of parts. Circuits are not static, they can modify themselves or be affected by other elements. In its simplest version, a circuit is just an <b>organized set of parts</b>. The organization method defines the circuit architecture and provides functionality. This organization can be established by any structure; whether it is a drawing or a fully-fledged software data structure.

                                The circuit <b>structure</b> to be used depends on the desired <b>functionality</b>, but at the same time conditions the behavior and relationships between the parts. The most standard structure is a simple one dimensional array and is what probably works best for most applications, since it is the closest to the biological organization of DNA. In this structure, circuits would be limited to adjacent parts, while other non-adjacent parts, circuits or genetic material somehow functionally related would be addressed as part of the environment. This helps set a clear limit to what a single circuit encompasses, since functional interactions with other genetic material can be infinite, and it would separate the circuit (synthetic DNA) from the organism's endogenous genetic material.

                                If a simple one dimensional structure doesn’t fit the functionality, a circuit can be defined using other structures, such as circular lists or hierarchical tree structures and graphs, if that suits the application best. Remember that the only requirement is that the circuit must be an organized container, the structure can vary.<br>
                            </p>


                        </div>


                        <div class="container  mx-auto p-3 wikicard-12-img">

                            <div class="text-center">
                                <img src="https://2021.igem.org/wiki/images/e/e6/T--TEC_COSTA_RICA--Description_Framework_07_CircuitDescription.png" alt="" style="width:100% center"
                                    class="w-75">
                            </div>
                        </div>


                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h3>Environment </h3>

                            <p class="">
                                A circuit can <b>interact with many different elements</b> apart from itself, such elements can modify the circuit  structurally, behaviorally or both. The environment describes <b>both the circuit and these simultaneous conditions</b> that potentially interact with or could affect it, but are not part of the circuit itself. The environment can be subdivided into circuits, <b>internal</b> (plasmid, genome, intracellular proteins) and <b>external</b> (media, environmental conditions).

                                Since this  framework is intended to be applied in a wide range of areas, some that we may not have yet considered the specific requirements for, it is especially important for the element of the environment to be a broad concept to act modularly to factors we may not have reckoned yet.<br>
                            </p>


                        </div>


                        <div class="container  mx-auto p-3 wikicard-12-img">

                            <div class="text-center">
                                <img src="https://2021.igem.org/wiki/images/9/9f/T--TEC_COSTA_RICA--Description_Framework_08_EnvironmentDescription.png" alt="" style="width:100% center"
                                    class="w-75">
                            </div>
                        </div>


                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2>Behavioral </h2>
                            <!-- <h3>subtitle </h3> -->

                            <p class="">
                                Since circuit state changes are allowed by the function of the circuit's parts and the environment, it is required to conceptualize how a part, multiple parts and the complete circuit act and interact with each other. The multiple ways in which parts relate with each other and the environment enables them to act, carry out their biological role, expand it, and inhibit other parts. These relationships can achieve high levels of complexity, especially since circuits can modify themselves, the environment and thus change states, requiring their constant validation and integrating dependent networks of interactions. The behavioral stage is still part of the description phase, thus, these are only the definitions of the actions; the process of carrying them out relates to the second phase, the execution.<br>
                            </p>

                            <h3>Interactions </h3>

                            <p class="">
                                Interactions describe how multiple <b>parts</b> and even <b>other elements of the environment influence each other</b>, affecting their transcription, translation or activity. These interactions are not mutually exclusive and may even overlap. Some of these relationships are:<br>
                            </p>

                        </div>
                        <div class="container w-75">

                            <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                                <div class="col-sm-6">
                                    <i>Alignment </i>
                                    <h3> </h3>
                                    <p>
                                        Parts can have a <b>directionality</b> relative to each other in the circuit, 

                                        <i>E.g.</i> Two recombinase recognition sites facing each other, facing away, or facing the same direction determine the carried out action; A unidirectional terminator´s direction relative to a coding part.<br>
                                    </p>
                                </div>

                                <div class="col-sm-6 justify-content-md-center">
                                    <img class="img-square align-middle "
                                        src="https://2021.igem.org/wiki/images/a/a0/T--TEC_COSTA_RICA--Description_Framework_09_InteractionAlignmentDescription.png" all="">
                                </div>
                            </div>

                            <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                                <div class="col-sm-6">
                                    <i>Position </i>
                                    <h3> </h3>
                                    <p>
                                        Parts have a <b>location</b> in the circuit relative to each other.

                                        <i>E.g.</i> a promoter upstream or downstream of a coding part.<br>
                                    </p>
                                </div>

                                <div class="col-sm-6 justify-content-md-center">
                                    <img class="img-square align-middle "
                                        src="https://2021.igem.org/wiki/images/1/12/T--TEC_COSTA_RICA--Description_Framework_10_InteractionPositionDescription.png" all="">
                                </div>
                            </div>

                            <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                                <div class="col-sm-6">
                                    <i>Co-existence </i>
                                    <h3> </h3>
                                    <p>
                                        <b>potentially interact</b> with each other when coexisting in the same environment.

                                        <i>E.g.</i> A recombinase and a RDF (recombinase directionality factor) can potentially interact to perform the inversion.<br>
                                    </p>
                                </div>

                                <div class="col-sm-6 justify-content-md-center">
                                    <img class="img-square align-middle "
                                        src="https://2021.igem.org/wiki/images/0/09/T--TEC_COSTA_RICA--Description_Framework_11_InteractionCoexistenceDescription.png" all="">
                                </div>
                            </div>

                            <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                                <div class="col-sm-6">
                                    <i>Co-dependence </i>
                                    <h3> </h3>
                                    <p>
                                        When two or more elements <b>require each other to function</b> there is a codependency between these, that would require for parts not only to be present, but also to be functional.

                                        <i>E.g.</i> A sgRNA requires a Cas protein for it to act; a recombinase requires at least two recognition sites<br>
                                    </p>
                                </div>

                                <div class="col-sm-6 justify-content-md-center">
                                    <img class="img-square align-middle "
                                        src="https://2021.igem.org/wiki/images/5/50/T--TEC_COSTA_RICA--Description_Framework_11_InteractionCodependenceDescription.png" all="">
                                </div>
                            </div>

                            <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                                <div class="col-sm-6">
                                    <i>Recognition </i>
                                    <h3> </h3>
                                    <p>
                                        This relationship describes any kind of <b>recognition</b> between two or more element, such as dna-protein, protein-protein, dna-rna, rna-protein 

                                        <i>E.g.</i> An inhibitor and an operator site; an RNA polymerase and the promoter; a protein and an associable subunit<br>
                                    </p>
                                </div>

                                <div class="col-sm-6 justify-content-md-center">
                                    <img class="img-square align-middle "
                                        src="https://2021.igem.org/wiki/images/a/ae/T--TEC_COSTA_RICA--Description_Framework_11_InteractionRecognitionDescription.png" all="">
                                </div>
                            </div>
                        </div>



                        <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                            <div class="col-sm-6">
                                <h3>Functions </h3>
                                <p>
                                    A function is an <b>action carried out by a part or group of parts determined by a set of interactions</b>. Some functions are: inversion, excision, edition, insertion, repression, expression.<br>


                                    For example, the function of a recombinase is determined by an interaction of alignment with a promoter for the coding sequence can be expressed, an interaction of codependency with the recognition sites, and an interaction of recognition of the sites.<br>
                                </p>
                            </div>

                            <div class="col-sm-6 justify-content-md-center">
                                <img class="img-square align-middle " src="https://2021.igem.org/wiki/images/d/da/T--TEC_COSTA_RICA--Description_Framework_12_FunctionDescription.png"
                                    all="">
                            </div>
                        </div>


                    </div>

                    <!-- Execution -->
                    <div v-if="current_slide_index['framework'].index==6" class="container">

                        <div class="accordion accordion-flush" id="accordionPanelsStayOpenExample2">
                            <div class="accordion-item">
                                <h2 class="accordion-header" id="panelsStayOpen-headingOne2">
                                    <button class="accordion-button" type="button" data-bs-toggle="collapse"
                                        data-bs-target="#panelsStayOpen-collapseOne2" aria-expanded="true"
                                        aria-controls="panelsStayOpen-collapseOne2">
                                        Aplication
                                    </button>
                                </h2>
                                <div id="panelsStayOpen-collapseOne2" class="accordion-collapse collapse show"
                                    aria-labelledby="panelsStayOpen-headingOne2">
                                    <div class="accordion-body">


                                        <div
                                            class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                                            <h2>Application </h2>
                                            <h3> </h3>

                                            <p class="">
                                                The application refers to how the <b>actions</b>, based on the <b>structure and relationships</b>, are <b>carried out</b>. This stage focuses on the execution of the parts’ functions from a previously defined circuit and its environment, as a processing workflow. Therefore, the <b>input</b> necessary to develop this workflow is a <b>defined environment</b> -which includes a circuit-, and the final <b>output</b> is <b>one or more modified circuits in their environments</b>. Multiple outputs might be necessary when considering alternative scenarios for the execution of the parts functions. One cycle of this workflow could encompass one or multiple circuit modifications, simultaneous or not, depending on the desired output.<br>
                                                <br>
                                                <h3>Processing workflow:</h3<
                                                
                                            </p>


                                            <p class="wiki-card-2columns">
                                                <ul>
                                                    <li><b>Define the potential interactions</b> of all elements. </li>
                                                    <li>Review the environment (excluding the circuit) and <b>define the elements that interact with the circuit</b> to consider them in this cycle of the workflow, the rest can be stored for a following cycle.</li>
                                                    <li>Traverse the circuit to <b>find the active promoters, terminators, operators and other regulatory parts</b> (<i>i.e.</i> can carry out their function of promoting or repressing transcription), based on their interactions with the environment or because they are constitutive.</li>
                                                    <li>Traverse the circuit starting from the promoter(s) and following the circuit’s structure (biologically: downstream) to review the <b>coding parts that can be transcribed</b> based on the alignment and position interactions with the parts determined in step 3.</li>
                                                    <li><b>Validate the interactions of the elements</b> (part(s) and other environment) to ensure the requirements for the function’s execution are met.</li>
                                                    <li>If necessary, define a <b>priority</b> to perform the functions or else, a simultaneity strategy.</li>
                                                    <li><b>Execute the elements functions</b> following the prioritization or considering as many scenarios as desired to generate one or multiple parallel new states of the circuit, depending on whether alternative scenarios are considered or not.</li>
                                                    <li><b>Review changes in the environment</b>, <i>e.g.</i> protein degradation, and extract one or multiple outputs per each modification</li>
                                                </ul>

                                            </p>
                                        </div>


                                        <div class="container  mx-auto p-3 wikicard-12-img">

                                            <div class="text-center">
                                                <img src="//placehold.co/1200x600?text=ANIMACION" alt=""
                                                    style="width:100% center" class="w-75">
                                            </div>
                                        </div>

                                    </div>
                                </div>
                            </div>








                            <div class="accordion-item">
                                <h2 class="accordion-header" id="panelsStayOpen-headingTwo2">
                                    <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
                                        data-bs-target="#panelsStayOpen-collapseTwo2" aria-expanded="false"
                                        aria-controls="panelsStayOpen-collapseTwo2">
                                        Integration
                                    </button>
                                </h2>
                                <div id="panelsStayOpen-collapseTwo2" class="accordion-collapse collapse"
                                    aria-labelledby="panelsStayOpen-headingTwo2">
                                    <div class="accordion-body">

                                        <div
                                            class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                                            <h2>Integration </h2>
                                            <h3> </h3>

                                            <p class="">
                                                The  process described above sets the foundation for operating over a circuit; this process can be the <b>base of larger more complex applications</b> by incorporating it into a larger workflow. This is what we call integration and it is an essential part of studying sequential circuits.

                                                This process of how to integrate a specific circuit cycle simulation is largely dependent on the application and can be used to incorporate new elements inline. By modifying a circuit through the processing workflow and then taking the output (the modified circuit and environment) as the new input, we are able to study <b>chains of events and their effects</b>. An alternative to the base cyclic approach would be to connect a <b>circuit generator</b> and a <b>post-execution circuit evaluator</b>; this way smarter selection approaches can be integrated into smarter the process such as <b>genetic algorithms</b> and <b>machine learning applications</b>, or see the <b>compatibility with a desired organism</b> by, for example, <b>analyzing the endogenous machinery</b> and potential <b>crosstalk</b>.<br>
                                            </p>
                                        </div>

                                        <div class="container  mx-auto p-3 wikicard-12-img">
                                            <div class="text-center">
                                                <img src="//placehold.co/1200x400?text=snippet" alt=""
                                                    style="width:100% center" class="w-75">
                                            </div>
                                        </div>

                                        <div class="container  mx-auto p-3 wikicard-12-img">
                                            <div class="text-center">
                                                <img src="//placehold.co/1200x400?text=snippet" alt=""
                                                    style="width:100% center" class="w-75">
                                            </div>
                                        </div>


                                        <div class="container  mx-auto p-3 wikicard-12-img">
                                            <div class="text-center">
                                                <img src="//placehold.co/1200x400?text=snippet" alt=""
                                                    style="width:100% center" class="w-75">
                                            </div>
                                        </div>


                                        <div class="container  mx-auto p-3 wikicard-12-img">
                                            <div class="text-center">
                                                <img src="//placehold.co/1200x400?text=snippet" alt=""
                                                    style="width:100% center" class="w-75">
                                            </div>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>








                    </div>

                    <!-- Software Framework -->
                    <div v-if="current_slide_index['framework'].index==7" class="container">

                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2> </h2>
                            <h3> </h3>

                            <p class="">
                                As mentioned before, the conceptual framework was a set of ideas to guide a computational or manual design, construction, simulation and analysis of synthetic gene circuits as state machines. We decided to apply our framework as a base for software tools’ design, where such tools would aid the exploration of alternative circuit architectures, while offering flexibility in the implementation and optimization. This means, we <b>implemented the concepts described and the coupling of them with sequential logic</b> -specially the {processing workflow}- <b>as computational concepts and a workflow to be used as a software’s guide.</b><br><br>


                                A software framework provides an <b>abstraction for generic software functionalities</b> upon which other software applications can be built. It makes <b>implementation details transparent to other developers</b>, which gets rid of most tedious and repetitive work, while providing a sturdy base for development and allowing most of the development effort to be tailored towards functional and valuable software. Thus this software framework can be easily used as a starting point for implementing software tools.<br><br>


                                The objective of this software framework is to set a <b>structural guide for facilitating the design of a software</b> to be used for the generation, assessment and manipulation of synthetic gene circuits as state machines. <br><br>


                                The workflow presented begins from the premise that the desired parts and their properties are defined. The software framework is divided into four: an initialization stage, two core processing stages and a post-processing stage.


                            </p>
                        </div>

                        <div class="container  mx-auto p-3 wikicard-12-img">
                            <div class="text-center">
                                <img src="https://2021.igem.org/wiki/images/d/d7/T--TEC_COSTA_RICA--Description_Framework_18_DiagramSoftwareFramework.png" alt="" style="width:100% center"
                                    class="w-75">
                            </div>
                        </div>


                        <div class="accordion accordion-flush" id="accordionPanelsStayOpenExample3">
                            <div class="accordion-item">
                                <h2 class="accordion-header" id="panelsStayOpen-headingOne3">
                                    <button class="accordion-button" type="button" data-bs-toggle="collapse"
                                        data-bs-target="#panelsStayOpen-collapseOne3" aria-expanded="true"
                                        aria-controls="panelsStayOpen-collapseOne3">
                                        Stage 1: Generator
                                    </button>
                                </h2>
                                <div id="panelsStayOpen-collapseOne3" class="accordion-collapse collapse show"
                                    aria-labelledby="panelsStayOpen-headingOne3">
                                    <div class="accordion-body">
                                       The initialization stage, which precedes the processing, is a circuit generator, where the environment with the circuit is defined. The circuit can be a preexisting one, built upon another one, or designed from scratch using a set of parts. The construction of the circuit can consider different levels of randomness and/or certain constraints, such as base pair length, number of parts, obligatory parts, number of promoters, etc. This stage can also tap into machine learning for a guided design of the circuits by, for example, clustering the generated outputs and using that data to generate artificial new circuits with similar characteristics. The chosen parameters for the circuit generation can direct the process to have a starting point more suitable for the specific objective.

                                        <strong>Input:</strong> defined parts with their corresponding properties.

                                        <strong>Output:</strong> a data structure that defines the circuit through its parts and order. <br>


                                        From a defined circuit, a cycle of two core stages is executed, this cycle processes the change from one state to another, <i>i.e.</i> the modification of the circuit architecture. A single cycle, or state change, consists of evaluation and action.<br>

                                    </div>
                                </div>
                            </div>



                            <div class="accordion-item">
                                <h2 class="accordion-header" id="panelsStayOpen-headingTwo3">
                                    <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
                                        data-bs-target="#panelsStayOpen-collapseTwo3" aria-expanded="false"
                                        aria-controls="panelsStayOpen-collapseTwo3">
                                        Stage 2: Evaluator
                                    </button>
                                </h2>
                                <div id="panelsStayOpen-collapseTwo3" class="accordion-collapse collapse  show "
                                    aria-labelledby="panelsStayOpen-headingTwo3">
                                    <div class="accordion-body">
                                       The evaluator traverses the circuit following its organization -given by the circuit's data structure- to define the parts that are functional, <i>i.e.</i> expressed, by considering relevant dependencies, such as repressors, inducers, promoters, terminators. The functionality relates to the interactions involving coding parts and other parts that promote or inhibit transcription, such as promoters, terminators, operators, recognition sites.

                                        <strong>Input:</strong> a data structure that defines the circuit through its parts and order.

                                        <strong>Output: A list of parts that get expressed in this state. The related inputs and interaction that enables the expression of the parts: which promoter and/or which inducer.

                                    </div>
                                </div>
                            </div>




                            <div class="accordion-item">
                                <h2 class="accordion-header" id="panelsStayOpen-headingThree3">
                                    <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
                                        data-bs-target="#panelsStayOpen-collapseThree3" aria-expanded="false"
                                        aria-controls="panelsStayOpen-collapseThree3">
                                        Stage 3: Actuator
                                    </button>
                                </h2>
                                <div id="panelsStayOpen-collapseThree3" class="accordion-collapse collapse show"
                                    aria-labelledby="panelsStayOpen-headingThree3">
                                    <div class="accordion-body">
                                        

                                        In the actuator phase, the functions of the previously tagged functional parts are executed. This process must follow the same organization and can consider simultaneous functions, alternate scenarios, or follow a single route. After the function is executed, a different circuit state is obtained. Therefore, the state change is processed by carrying out the functionality.

                                        <strong>Input:</strong> a data structure that defines the circuit through its parts and order, and a list of parts that get expressed in this state.

                                        <strong>Output:</strong> a new state or states as data structures that define each circuit.<br>

                                        At this point, the output circuit or circuits can be cycled back into the validator to redefine the functional parts. Thus, the number of evaluator/actuator cycles is repeated according to the number of desired states/state changes, or until there are no more possible modifications.<br>

                                    </div>
                                </div>
                            </div>


                            <div class="accordion-item">
                                <h2 class="accordion-header" id="panelsStayOpen-headingForth3">
                                    <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
                                        data-bs-target="#panelsStayOpen-collapseForth3" aria-expanded="false"
                                        aria-controls="panelsStayOpen-collapseForth3">
                                        Stage 4: Analysis
                                    </button>
                                </h2>
                                <div id="panelsStayOpen-collapseForth3" class="accordion-collapse collapse show"
                                    aria-labelledby="panelsStayOpen-headingForth3">
                                    <div class="accordion-body">
                                        After the desired set of cycles are runned, or throughout it, the analysis stage is performed. The analysis is based on a database fed the designed and modelled circuits. Everytime a evaluator/actuator cycle is performed, the generated state or states are stored here, thus saving the history of the circuit’s behavior. Through the database, the circuits can be analyzed and filtered using simple queries based on the circuit’s characteristics. The filters allow for the development and exploration of circuits to be oriented.<br>
                                    </div>
                                </div>
                            </div>



                        </div>






                    </div>

                    <!-- Conceptual Framework APPLIED TO DEVELOP THE SOFTWARE FRAMEWORK -->
                    <div v-if="current_slide_index['framework'].index==8" class="container">


                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2>Description </h2>
                            <!-- <h3> </h3> -->

                            <!-- <p class="wiki-card-2columns">
                                </p> -->
                        </div>

                        <div class="container wiki-card mx-auto p-3 row wikicard-6-2-img-no-number">
                            <div class="col-sm-6">
                                <h3>Structural </h3>
                                <p>
                                    A circuit seen as an organized set of the parts is implemented as a data structure, where this data structure provides the specific organization.
                                </p>
                            </div>

                            <div class="col-sm-6">
                                <h3>Behavioral </h3>
                                <p>
                                    The interactions and functions of parts and other biological elements are applied as functions that each part can call on the validator and actuator, or as information. Alignment interactions define the functional parts in the validator. Other interactions are considered for the actuator to generate the changes the circuit will go through, whether as requirements or as function itself.
                                </p>
                            </div>


                        </div>


                        <div class="container wiki-card wikicard-12 mx-auto p-3 row  wikicard-12-double-col">
                            <h2>Execution: application/integration </h2>
                            <!-- <h3> </h3> -->

                            <p class="wiki-card-2columns">
                                The software framework is the direct application of the execution phase. The generator-validator stages execute the part's functions as in the atomic workflow, enabling the modelling of the circuits as state machines. On the other hand, as the software framework's objective is the design of circuits and the exploration of the design space, the generator-validator circuit modelling requires the integration of other stages that would complement it. This was implemented as the first and fourth stages, generator and analysis, which coupled enable the building, saving and analisis the circuits. <br>
                            </p>
                        </div>






                    </div>


                </div>
                <!-- pills-framework -->


            </div>
        </div>
        <!-- DEMO TABS -->















    </div>

</body>
<!-- CONTENIDO PRINCIPAL -->

<style>

</style>

<style>
    #globalWrapper {
        font-size: 100% !important;
    }

    #HQ_page h1,
    h2,
    h3,
    h4,
    h5 {
        font-family: var(--bs-body-font-family) !important;
        overflow: visible !important
    }

    #HQ_page p {
        font-family: var(--bs-body-font-family) !important;
        font-size: var(--bs-body-font-size) !important;
    }



    /*
    #HQ_page h1,
    h2,
    h3,
    h4,
    h5 {
     font-family:"Arial",Gadget,sans-serif;
     border-bottom:none;
     font-weight:bold
    }

    #HQ_page p {
        font-family:"Arial",Helvetica,sans-serif;
        font-size:13px;
        text-align:justify
       }
       
    /*body {
        font-family: var(--bs-body-font-family) !important;
    }*/

    .double-text {
        background-color: #1A1A1A;
        -webkit-text-stroke-width: 1.5px;
        -webkit-text-fill-color: rgba(255, 255, 255, 0);
        -webkit-text-stroke-color: rgb(255, 255, 255);
        font-size: 90px;
        font-family: 'Open Sans';
        font-weight: 800 !important;

        text-shadow:
            6px 8px 0px rgb(255, 255, 255);
        opacity: 1;
        text-transform: uppercase;
        letter-spacing: -1.03px;
    }


    .main-title h1 {
        /*font-family: 'Open Sans'; */
        font-weight: 800 !important;
        font-size: 55px;
        -webkit-text-fill-color: #ffffff00;
        -webkit-text-stroke-width: 1.5px;
        -webkit-text-stroke-color: #ffffff;
    }


    .main-title h5 {
        /*font-family: 'Open Sans'; */
        font-size: 15px;
        color: #ffffff;
        width: 40%
    }

    .main-title-bg {
        opacity: 0.9;
        position: absolute;
        left: 0;
        top: 0;
        width: 100%;
        height: auto;
        z-index: -99;
    }

    .wiki-card {}

    .wiki-card .img-round {
        border-radius: 50%;
        width: 100%;
        height: auto;

    }

    .wiki-card .img-square {
        border-radius: 10px;
        width: 100%;
        height: auto;
    }

    .wiki-card h1 {
        /*margin-right: 10vw;*/
        color: #E2474B;
        font-size: calc(2rem + 1.5vw);
        /*font-size: 56px;*/
        font-weight: bold;
        letter-spacing: -0.8px;
    }

    .wiki-card h2 {
        color: #27262D;
        font-size: 24px;
        font-weight: bold;
        letter-spacing: 0;
        line-height: 33px;
    }

    .wiki-card h3 {
        color: #626262;
        font-size: 18px;
        letter-spacing: -0.2px;
        line-height: 19px;
    }

    .wiki-card p {
        color: #27262D;
        /*line-height: 24px;*/
    }

    .wiki-card p.wiki-card-2columns {
        columns: 2;
    }


    .wiki-tab-btn {
        background-color: #ffffff20 !important;
        border-color: #ffffff00 !important;
        border-radius: 0px;
        margin: 0px;
        padding: .2rem .5rem;
        color: #ffffff;
        font-size: 20px;
        font-family: 'Open Sans';

    }

    .wiki-tab-btn.active {
        border-bottom: 10px solid rgb(255, 0, 0) !important;
        color: #ffffff;
    }

    .wiki-tab-btn:hover {
        color: #ffffff;
    }

    /*button:focus{
        background-color:olive!important;
    }*/
</style>


<script>
    var app = new Vue({
        el: '#app',
        data() {
            return {
                isActive: true,
                message: 'hello app',
                current_slide_index: {
                    "circuit": {
                        index: 0
                    },
                    "framework": {
                        index: 0
                    }
                },
                slides_content: {
                    "circuit": [{
                            title: "Introduction",
                            content: "Our project began with an objective in mind: expanding the boundaries of engineered organisms, specifically targeting their release. The design and implementation of a genetic suicide circuit is the beginning of this journey."
                        },
                        {
                            title: "Function",
                            content: "Our circuit’s behavior is directly related to four <b>separate modules</b>. Each of these exhibits a specific function in order to fulfill a determined purpose. Here we propose a coordinated behavior; nonetheless they can also be used separately. "
                        },
                        {
                            title: "Architecture and theoretical basis",
                            content: "As for any well designed genetic circuit we took great care and consideration for the selection of the different parts and proteins included in our circuit. In this case, architecture is of high relevance, since it’s the key to allow the behavior and function we described earlier. This is even truer for our counter circuit, which we conceptualized as a state machine< and, as such, defines each state by specific DNA configurations and requires a specific input to change state."
                        },
                        {
                            title: "Characteristics",
                            content: "To showcase some of the best qualities of our circuit, we handpicked some of its characteristics of interest, most of them brought forward by our user interviews."
                        },
                        {
                            title: "Functional Animation",
                            content: "To better understand the behavior of our circuit we have developed an interactive animation in which each module can be activated and explored as their different steps unfold."
                        },
                        {
                            title: "Extras",
                            content: "This section contains additional modules along with variations for the ones presented within our main circuit. Their intention is to improve functionality and expand the utilization possibilities."
                        },
                        {
                            title: "Comparison",
                            content: "To further demonstrate the novelty and capacity of our counter circuit we present a comparison with previously developed counters, highlighting their characteristics of interest."
                        },
                        {
                            title: "References",
                            content: ""
                        }
                    ],
                    "framework": [{
                            title: " Introduction",
                            content: "How the framework came to be? who are our users?"
                        },
                        {
                            title: "Project description",
                            content: "What actually is our conceptual framework?"
                        },
                        {
                            title: "Foundational Concepts",
                            content: "This framework is supported by three key concepts: sequential logic, abstraction and modularity."
                        },
                        {
                            title: "Framework's architecture",
                            content: "How our guide is organized"
                        },
                        {
                            title: "Biological state machines",
                            content: "The section Biological state machines encompasses an expansion of how a circuit can be expressed as a state machine and the characteristics such a system has."
                        },
                        {
                            title: "Description",
                            content: "The description phase entails the conceptualization of biological objects and how they interact with one another to carry out complex processes. Here we provide thorough definitions for each concept."
                        },
                        {
                            title: "Execution",
                            content: "The second phase -the execution phase- entails the union of the concepts previously defined, both the structural parts and the interactions between them, and sequential logic to describe the design, and model the changes of states of a genetic circuit. Here we propose the complete workflows for this process."
                        },
                        {
                            title: "Software framework",
                            content: "The implementation of the conceptual framework as a guide for software design"
                        },
                        {
                            title: "Conceptual Framework Applied To Develop The Software Framework",
                            content: "As mentioned before, the software framework is a direct implementation of the conceptual framework, by applying the concepts described and the coupling of them with sequential logic as computational concepts and a workflow."
                        }
                    ]
                },
            }
        },
        methods: {},
        mounted() {}
    })
</script>


</html>
